<html><head>
<meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">

<!--Arlequin results file-->


<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head><body bgcolor="white">

<pre><a name="18_09_11at16_08_26">&nbsp;</a>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (18/09/11 at 16:08:26)
////////////////////////////////////////////////////////////////////

 #[WARNING # 1] : sum of genotype frequencies do not match specified sample size : adjusting sample size
<a name="18_09_11at16_08_26_run_information">&nbsp;</a>

Project information:
--------------------
	NbSamples      =  14
	DataType       =  MICROSAT
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.3

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :No. of different alleles (FST)
        GammaA Value       = 0
        Theta estimators   : 

    Linkage disequilibrium test:
    ----------------------------
        No. of permutations          = 10000
        No. of Initial conditions    = 2
        Print Histogram and table : Yes
        Significance level        = 0.05

    Hardy-Weinberg equilibrium test:
    --------------------------------
        No. of steps in Markov chain      = 1000000
        No. of Dememorisation Steps       = 100000
        Required precision on Probability = 0
        Test association at the Locus level

    Analysis of Molecular Variance:
    -------------------------------
        No. of Permutations = 1000

    Population pairwise Fst values:
    -------------------------------
        Compute coancestry coefficients
        Compute Slatkin's distances
        No. of permutations for significance = 100
        No. of permutations for Mantel test  = 1000

       Distance matrix:
            Compute distance matrix
            Molecular distance : No. of different alleles (FST)
            Gamma a value      = 0

    Exact test of population differentiation:
    -----------------------------------------
        No. of steps in Markov chain      = 100000
        No. of Dememorisation Steps       = 10000
        Required precision on Probability = 0
        Print Histogram and table : Yes
        Significance level        = 0.05


    Assignement of individual genotypes to populations
    --------------------------------------------------


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 1000
<a name="18_09_11at16_08_26_gen_struct">&nbsp;</a>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 5
Allowed level of missing data                  : 0.3

List of usable loci :
---------------------
   1     2     3     4     5  

List of loci with too much missing data :
-----------------------------------------

NONE

<a name="18_09_11at16_08_26_amova">&nbsp;</a>

=================================================================
  AMOVA ANALYSIS
=================================================================


---------------------------
Genetic structure to test :
---------------------------

No. of Groups = 7


	[[Structure]]

		StructureName = "New Edited Structure"
		NbGroups = 7
		Group={
		"QTUR-2"
		"QTUR-1"
		}

		Group={
		"QRL2-1"
		"QRL2-2"
		}

		Group={
		"QRL1-2"
		"QRL1-1"
		}

		Group={
		"QLL3-2"
		"QLL3-1"
		}

		Group={
		"QLL1-2"
		"QLL1-1"
		}

		Group={
		"QBED2-2"
		"QBED2-1"
		}

		Group={
		"QBED1-2"
		"QBED1-1"
		}

---------------------------


Distance method: No. of different alleles (FST)

--------------------------
AMOVA design and results :
--------------------------

Reference: <i>Weir, B.S. and Cockerham, C.C. 1984.</i>
           <i>Excoffier, L., Smouse, P., and Quattro, J. 1992.</i>
           <i>Weir, B. S., 1996.</i>
----------------------------------------------------------------------
 Source of                  Sum of      Variance         Percentage
 variation      d.f.        squares     components       of variation
----------------------------------------------------------------------
 Among
 groups           6         24.668        0.01601 Va             1.04

 Among
 populations
 within
 groups           7         19.812        0.02679 Vb             1.74

 Among
 individuals
 within
 populations    266        467.875        0.25804 Vc            16.72
 Within
 individuals    280        348.000        1.24286 Vd            80.51
----------------------------------------------------------------------
 Total          559        860.355        1.54369
----------------------------------------------------------------------
 Fixation Indices
      FIS :      0.17192
      FSC :      0.01753
      FCT :      0.01037
      FIT :      0.19488
----------------------------------------------------------------------

Significance tests (1023 permutations)
------------------

Vd and FIT : P(rand. value &lt; obs. value)  =  0.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.00000+-0.00000

Vc and FIS : P(rand. value &gt; obs. value)  =  0.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.00000+-0.00000

Vb and FSC : P(rand. value &gt; obs. value)  =  0.00000
             P(rand. value = obs. value)  =  0.00098
                                  P-value =  0.00098+-0.00098

Va and FCT : P(rand. value &gt; obs. value)  =  0.08016
             P(rand. value = obs. value)  =  0.00098
                                  P-value =  0.08113+-0.00848

<a name="18_09_11at16_08_26_amova_AMOVA_FIS">&nbsp;</a>

-------------------------------------------------------
Population specific FIS indices (1023 permutations)
---------------------------------------------------
  Pop#      Name       FIS    P(Rand FIS&gt;=Obs FIS)
---------------------------------------------------
     1    QTUR-2   0.29035            0.000000
     2    QTUR-1   0.13559            0.024438
     3    QRL2-1   0.13821            0.018573
     4    QRL2-2   0.25717            0.000000
     5    QRL1-2   0.13077            0.022483
     6    QRL1-1   0.26039            0.001955
     7    QLL3-2   0.30391            0.000000
     8    QLL3-1   0.24161            0.000000
     9    QLL1-2   0.17244            0.006843
    10    QLL1-1   0.06647            0.166178
    11   QBED2-2   0.07317            0.137830
    12   QBED2-1   0.26328            0.000000
    13   QBED1-2   0.08685            0.088954
    14   QBED1-1  -0.03951            0.767351
---------------------------------------------------

<a name="18_09_11at16_08_26_pairw_diff">&nbsp;</a>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	QTUR-2
  2:	QTUR-1
  3:	QRL2-1
  4:	QRL2-2
  5:	QRL1-2
  6:	QRL1-1
  7:	QLL3-2
  8:	QLL3-1
  9:	QLL1-2
  10:	QLL1-1
  11:	QBED2-2
  12:	QBED2-1
  13:	QBED1-2
  14:	QBED1-1

------------------------
Population pairwise FSTs
------------------------


Distance method: No. of different alleles (FST)
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1   0.00000
           2   0.03558   0.00000
           3   0.01030   0.06555   0.00000
           4   0.02040   0.06375   0.00989   0.00000
           5   0.02439   0.07921   0.00163   0.03742   0.00000
           6   0.02530   0.07296   0.02761   0.03937   0.02784   0.00000
           7   0.02695   0.05658   0.05398   0.05754   0.05699   0.04898   0.00000
           8   0.00715   0.00611   0.04006   0.03664   0.05001   0.03001   0.01791   0.00000
           9   0.01657   0.08013  -0.01162   0.01851  -0.00266   0.01504   0.05520   0.04125   0.00000
          10   0.00739   0.04922   0.01555   0.03821   0.02219   0.03780   0.05419   0.03166   0.01700   0.00000
          11   0.00869   0.05031  -0.00158   0.01934   0.00201   0.03400   0.02254   0.02349   0.00004   0.00379   0.00000
          12   0.02471   0.08778   0.02302   0.02752   0.01722   0.03261   0.02196   0.03821   0.02161   0.05612   0.01632   0.00000
          13   0.02083   0.08518  -0.00357   0.03058  -0.00986   0.02546   0.03624   0.05025  -0.00586   0.03334   0.00741   0.00313   0.00000
          14   0.01605   0.08942  -0.00139   0.04528   0.02470   0.04380   0.08882   0.05637   0.00606   0.01490   0.01523   0.06290   0.02986   0.00000


------------
FST P values
------------

Number of permutations : 110

                       1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13                14
           1           *
           2   0.00901+-0.0091           *
           3   0.27027+-0.0429   0.00000+-0.0000           *
           4   0.17117+-0.0370   0.00000+-0.0000   0.24324+-0.0385           *
           5   0.09910+-0.0163   0.00000+-0.0000   0.40541+-0.0493   0.01802+-0.0121           *
           6   0.16216+-0.0227   0.00000+-0.0000   0.07207+-0.0227   0.01802+-0.0121   0.03604+-0.0201           *
           7   0.10811+-0.0297   0.00000+-0.0000   0.00901+-0.0091   0.00000+-0.0000   0.00000+-0.0000   0.00000+-0.0000           *
           8   0.49550+-0.0412   0.45946+-0.0497   0.00000+-0.0000   0.02703+-0.0139   0.00000+-0.0000   0.02703+-0.0139   0.22523+-0.0389           *
           9   0.15315+-0.0526   0.00000+-0.0000   0.93694+-0.0151   0.11712+-0.0237   0.66667+-0.0639   0.18919+-0.0316   0.00000+-0.0000   0.00000+-0.0000           *
          10   0.30631+-0.0278   0.00000+-0.0000   0.16216+-0.0379   0.01802+-0.0121   0.08108+-0.0212   0.00901+-0.0091   0.00901+-0.0091   0.00901+-0.0091   0.15315+-0.0237           *
          11   0.27928+-0.0438   0.00000+-0.0000   0.58559+-0.0245   0.09910+-0.0286   0.45045+-0.0359   0.03604+-0.0242   0.03604+-0.0148   0.01802+-0.0121   0.50450+-0.0578   0.45045+-0.0429           *
          12   0.04505+-0.0203   0.00000+-0.0000   0.01802+-0.0121   0.04505+-0.0152   0.20721+-0.0470   0.02703+-0.0139   0.13514+-0.0279   0.01802+-0.0121   0.07207+-0.0264   0.00000+-0.0000   0.10811+-0.0297           *
          13   0.09910+-0.0212   0.00000+-0.0000   0.77477+-0.0412   0.00901+-0.0091   0.94595+-0.0154   0.06306+-0.0139   0.00000+-0.0000   0.00000+-0.0000   0.85586+-0.0340   0.00901+-0.0091   0.26126+-0.0478   0.48649+-0.0433           *
          14   0.11712+-0.0273   0.00000+-0.0000   0.55856+-0.0379   0.00000+-0.0000   0.03604+-0.0148   0.02703+-0.0139   0.00000+-0.0000   0.00000+-0.0000   0.33333+-0.0678   0.09910+-0.0417   0.04505+-0.0203   0.00000+-0.0000   0.01802+-0.0121           *


------------
Matrix of significant Fst P values
Significance Level=0.0500
------------

Number of permutations : 110

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1                   +         -         -         -         -         -         -         -         -         -         +         -         -
           2         +                   +         +         +         +         +         -         +         +         +         +         +         +
           3         -         +                   -         -         -         +         +         -         -         -         +         -         -
           4         -         +         -                   +         +         +         +         -         +         -         +         +         +
           5         -         +         -         +                   +         +         +         -         -         -         -         -         +
           6         -         +         -         +         +                   +         +         -         +         +         +         -         +
           7         -         +         +         +         +         +                   -         +         +         +         -         +         +
           8         -         -         +         +         +         +         -                   +         +         +         +         +         +
           9         -         +         -         -         -         -         +         +                   -         -         -         -         -
          10         -         +         -         +         -         +         +         +         -                   -         +         +         -
          11         -         +         -         -         -         +         +         +         -         -                   -         -         +
          12         +         +         +         +         -         +         -         +         -         +         -                   -         +
          13         -         +         -         +         -         -         +         +         -         +         -         -                   +
          14         -         +         -         +         +         +         +         +         -         -         +         +         +          

---------------------------------------------------
Matrix of coancestry coefficients as t/M=-ln(1-FST)
(M=N for haploid data, M=2N for diploid data)
---------------------------------------------------

Reference: <i>Reynolds, J., Weir, B.S., and Cockerham, C.C. 1983.</i>
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1  -0.00000
           2   0.03623  -0.00000
           3   0.01036   0.06780  -0.00000
           4   0.02061   0.06587   0.00994  -0.00000
           5   0.02469   0.08253   0.00163   0.03813  -0.00000
           6   0.02562   0.07576   0.02800   0.04017   0.02824  -0.00000
           7   0.02732   0.05824   0.05549   0.05926   0.05868   0.05022  -0.00000
           8   0.00718   0.00613   0.04089   0.03732   0.05130   0.03047   0.01808  -0.00000
           9   0.01671   0.08352         0   0.01868         0   0.01515   0.05678   0.04213  -0.00000
          10   0.00741   0.05047   0.01567   0.03896   0.02244   0.03853   0.05572   0.03217   0.01715  -0.00000
          11   0.00873   0.05162         0   0.01953   0.00202   0.03459   0.02280   0.02377   0.00004   0.00380  -0.00000
          12   0.02502   0.09188   0.02329   0.02790   0.01737   0.03315   0.02220   0.03896   0.02185   0.05776   0.01645  -0.00000
          13   0.02105   0.08902         0   0.03106         0   0.02579   0.03691   0.05155         0   0.03391   0.00744   0.00314  -0.00000
          14   0.01618   0.09367         0   0.04634   0.02501   0.04478   0.09301   0.05802   0.00608   0.01501   0.01535   0.06496   0.03032  -0.00000


----------------------------------------------------
Matrix of Slatkin linearized FSTs as t/M=FST/(1-FST)
(M=N for haploid data, M=2N for diploid data)
----------------------------------------------------

Reference: <i>Slatkin, M., 1995. </i>
                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1   0.00000
           2   0.03689   0.00000
           3   0.01041   0.07015   0.00000
           4   0.02083   0.06809   0.00999   0.00000
           5   0.02499   0.08603   0.00163   0.03887   0.00000
           6   0.02595   0.07870   0.02839   0.04099   0.02864   0.00000
           7   0.02769   0.05997   0.05706   0.06105   0.06043   0.05151   0.00000
           8   0.00720   0.00615   0.04174   0.03803   0.05264   0.03094   0.01824   0.00000
           9   0.01685   0.08711         0   0.01886         0   0.01527   0.05842   0.04303   0.00000
          10   0.00744   0.05176   0.01580   0.03972   0.02270   0.03928   0.05730   0.03269   0.01730   0.00000
          11   0.00877   0.05298         0   0.01972   0.00202   0.03519   0.02306   0.02406   0.00004   0.00380   0.00000
          12   0.02534   0.09623   0.02357   0.02830   0.01752   0.03371   0.02245   0.03973   0.02209   0.05946   0.01659   0.00000
          13   0.02127   0.09311         0   0.03155         0   0.02613   0.03760   0.05290         0   0.03449   0.00747   0.00314   0.00000
          14   0.01631   0.09820         0   0.04743   0.02532   0.04580   0.09747   0.05973   0.00610   0.01513   0.01547   0.06712   0.03078   0.00000


------------------------------------------------------------------
Matrix of M values (M=Nm for haploid data, M=2Nm for diploid data)
------------------------------------------------------------------

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1
           2  13.55211
           3  48.03489   7.12785
           4  24.00621   7.34352  50.04754
           5  20.00429   5.81204 307.09677  12.86338
           6  19.26651   6.35333  17.61139  12.19929  17.45897
           7  18.05603   8.33744   8.76325   8.19036   8.27349   9.70771
           8  69.42308  81.29870  11.98000  13.14784   9.49841  16.16086  27.41306
           9  29.67992   5.74000       inf  26.51070       inf  32.74343   8.55803  11.62032
          10  67.19392   9.65909  31.65382  12.58686  22.02792  12.72898   8.72627  15.29293  28.90921
          11  57.00924   9.43818       inf  25.35805 247.70370  14.20735  21.68192  20.7822112362.50000 131.48760
          12  19.73526   5.19582  21.21750  17.67045  28.54125  14.83274  22.26854  12.58537  22.63666   8.40877  30.14286
          13  23.50598   5.37018       inf  15.84868       inf  19.13672  13.29787   9.45124       inf  14.49570  66.95251 159.05660
          14  30.65295   5.09162       inf  10.54217  19.74411  10.91660   5.12956   8.37034  82.02844  33.05489  32.32365   7.44941  16.24271



********************
Locus by locus AMOVA:
********************

Distance method for locus-by-locus analysis: No. of different alleles (FST)

List of loci with only one allele:
----------------------------------


<a name="18_09_11at16_08_26Loc_by_Loc_AMOVA">&nbsp;</a>

----------------------------------------
AMOVA Results for polymorphic loci only:
----------------------------------------

            Among Groups:                                      Among Population:                                    Within Populations:                                    Within Individuals:                                 Fixation indices:
  Locus            SSD   d.f.             Va    % variation            SSD   d.f.             Vb    % variation            SSD   d.f.             Vc    % variation            SSD   d.f.             Vd    % variation       FIS   P-value       FSC   P-value       FCT   P-value       FIT   P-value
      1        3.11250      6        0.00258        1.27126        2.18750      7        0.00187        0.92107       63.25000    266        0.03943       19.44104       44.50000    280        0.15893       78.36663   0.19877   0.00000   0.00933   0.15445   0.01271   0.08211   0.21633   0.00000
      2        4.40536      6        0.00114        0.34938        4.50000      7        0.00663        2.02905      100.42500    266        0.05841       17.86830       73.00000    280        0.26071       79.75327   0.18304   0.00000   0.02036   0.05181   0.00349   0.39198   0.20247   0.00000
      3        8.22249      6        0.00515        1.22368        6.75791      7        0.01205        2.86489      128.49415    261        0.08888       21.13143       86.50000    275        0.31455       74.78000   0.22032   0.00000   0.02900   0.00098   0.01224   0.23558   0.25220   0.00000
      4        3.94172      6        0.00295        0.95352        2.95735      7        0.00162        0.52299       94.56316    264        0.05320       17.18452       70.00000    278        0.25180       81.33897   0.17442   0.00000   0.00528   0.29326   0.00954   0.20332   0.18661   0.00000
      5        5.41916      6        0.00380        1.22827        4.21174      7        0.00684        2.20979       87.06765    263        0.03195       10.31597       74.00000    277        0.26715       86.24598   0.10683   0.00196   0.02237   0.01662   0.01228   0.21017   0.13754   0.00000

====================================================
Global AMOVA results as a weighted average over loci
====================================================

----------------------------------------------------
AMOVA design and results (average over 5 loci):
----------------------------------------------------

------------------------------------------------------------------------
 Source of               Sum of           Variance            Percentage
 variation               squares          components          variation
------------------------------------------------------------------------
 Among
 groups                   25.101             0.01562             0.99537

 Among
 populations
 within
 groups                   20.615             0.02902             1.84852

 Among
 individuals
 within
 populations             473.800             0.27187            17.32072
 Within
 individuals             348.000             1.25313            79.83539
------------------------------------------------------------------------
 Total                   867.516             1.56965
------------------------------------------------------------------------


Average F-Statistics over all loci
 Fixation Indices
      FIS :      0.17828
      FSC :      0.01867
      FCT :      0.00995
      FIT :      0.20165

------------------
Significance tests (1023 permutations)
------------------

Vd and FIT : P(rand. value &lt; obs. value)  =  0.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.00000

Vc and FIS:  P(rand. value &gt; obs. value)  =  0.00000
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.00000

Vb and FSC : P(rand. value &gt; obs. value)  =  0.00098
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.00098

Va and FCT : P(rand. value &gt; obs. value)  =  0.05670
             P(rand. value = obs. value)  =  0.00000
                                  P-value =  0.05670

<a name="18_09_11at16_08_26_comp_sum_LBL_AMOVA_FIS">&nbsp;</a>


-----------------------------------------------------------------------
Population specific FIS indices per polymorphic locus (absolute values)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
  Locus   Average FIS    QTUR-2    QTUR-1    QRL2-1    QRL2-2    QRL1-2    QRL1-1    QLL3-2    QLL3-1    QLL1-2    QLL1-1   QBED2-2   QBED2-1   QBED1-2   QBED1-1
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
      1       0.19877   0.00328  -0.03323   0.31940   0.31034   0.09953  -0.03636   0.43911   0.38874   0.29368  -0.08057   0.10705   0.36490   0.35955  -0.02395
      2       0.18304   0.50968   0.10471   0.17391   0.21198   0.02756   0.08333   0.48450   0.12240   0.22160   0.16940   0.15411   0.12917  -0.04000   0.18455
      3       0.22032   0.10938   0.24820   0.26119   0.23746   0.16413   0.61509   0.15287   0.57166   0.04762   0.18280  -0.05212   0.49558   0.12442  -0.12500
      4       0.17442   0.32411   0.14533   0.05430   0.32294   0.43162   0.39490   0.25311   0.07505   0.21851  -0.05069   0.04264   0.29368   0.09703  -0.25436
      5       0.10683   0.54354   0.13418  -0.02703   0.28889  -0.05751   0.07128   0.32277  -0.04828   0.22131  -0.05556   0.13992  -0.01527   0.05506   0.08800
-----------------------------------------------------------------------------------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------------------
Population specific FIS indices per polymorphic locus (relative values (%) compared to average FIS)
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
  Locus   Average FIS    QTUR-2    QTUR-1    QRL2-1    QRL2-2    QRL1-2    QRL1-1    QLL3-2    QLL3-1    QLL1-2    QLL1-1   QBED2-2   QBED2-1   QBED1-2   QBED1-1
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
      1       0.19877   -98.350  -116.719    60.691    56.134   -49.929  -118.295   120.918    95.575    47.750  -140.534   -46.143    83.582    80.890  -112.050
      2       0.18304   178.457   -42.792    -4.984    15.814   -84.943   -54.472   164.699   -33.127    21.070    -7.451   -15.804   -29.431  -121.854     0.827
      3       0.22032   -50.357    12.654    18.551     7.778   -25.505   179.179   -30.617   159.465   -78.387   -17.033  -123.655   124.932   -43.526  -156.735
      4       0.17442    85.821   -16.679   -68.869    85.150   147.462   126.409    45.116   -56.971    25.277  -129.063   -75.556    68.375   -44.370  -245.829
      5       0.10683   408.780    25.596  -125.298   170.413  -153.836   -33.276   202.129  -145.188   107.157  -152.002    30.969  -114.291   -48.459   -17.628
-----------------------------------------------------------------------------------------------------------------------------------------------------------------

<a name="18_09_11at16_08_26_exct_tests">&nbsp;</a>

================================================================================
== Exact Test of Sample Differentiation Based on Haplotype Frequencies
================================================================================
Reference: <i>Raymond M. and F. Rousset. 1995 .</i>
           <i>Goudet, J., M. Raymond, T. de Meeüs and F. Rousset, 1996.</i>

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1	QBED1-1
  2	QBED1-2
  3	QBED2-1
  4	QBED2-2
  5	QLL1-1
  6	QLL1-2
  7	QLL3-1
  8	QLL3-2
  9	QRL1-1
  10	QRL1-2
  11	QRL2-1
  12	QRL2-2
  13	QTUR-1
  14	QTUR-2


--------------------------------------------------------------------------
Global test of differentiation among sample:
--------------------------------------------

 Non-differentiation: Exact P value = 0.17264 +- 0.08652  (100000 Markov steps done)

--------------------------------------------------------------------------
Differentiation test between all pairs of samples:
--------------------------------------------------

Markov chain length : 100000 steps)

Non-differentiation exact P values :

                1                 2                 3                 4                 5                 6                 7                 8                 9                10                11                12                13

           2   1.00000+-0.0000
           3   1.00000+-0.0000   1.00000+-0.0000
           4   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000
           5   0.50282+-0.0188   0.47657+-0.0126   0.48818+-0.0189   0.47610+-0.0112
           6   0.49696+-0.0150   0.51417+-0.0122   0.47695+-0.0145   0.48878+-0.0178   0.24324+-0.0105
           7   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.48854+-0.0215   0.48450+-0.0077
           8   0.48480+-0.0086   0.48944+-0.0120   0.74661+-0.0073   0.51101+-0.0113   0.25888+-0.0123   0.23688+-0.0064   0.49842+-0.0179
           9   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49381+-0.0155   0.49027+-0.0084   1.00000+-0.0000   0.48646+-0.0118
          10   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49867+-0.0094   0.47911+-0.0182   1.00000+-0.0000   0.49678+-0.0152   1.00000+-0.0000
          11   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.47942+-0.0166   0.48910+-0.0095   1.00000+-0.0000   0.51051+-0.0115   1.00000+-0.0000   1.00000+-0.0000
          12   0.49843+-0.0164   0.48196+-0.0087   0.48531+-0.0147   0.50386+-0.0152   0.24915+-0.0080   0.60758+-0.0103   0.49776+-0.0169   0.25092+-0.0081   0.49333+-0.0125   0.48124+-0.0177   0.48580+-0.0121
          13   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.47201+-0.0159   0.49550+-0.0161   1.00000+-0.0000   0.48328+-0.0125   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.49764+-0.0119
          14   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.48402+-0.0113   0.47913+-0.0197   1.00000+-0.0000   0.48251+-0.0153   1.00000+-0.0000   1.00000+-0.0000   1.00000+-0.0000   0.48504+-0.0102   1.00000+-0.0000


--------------------------------
Table of significant differences (significance level=0.0500):
--------------------------------

                     1         2         3         4         5         6         7         8         9        10        11        12        13        14
           1                   -         -         -         -         -         -         -         -         -         -         -         -         -
           2         -                   -         -         -         -         -         -         -         -         -         -         -         -
           3         -         -                   -         -         -         -         -         -         -         -         -         -         -
           4         -         -         -                   -         -         -         -         -         -         -         -         -         -
           5         -         -         -         -                   -         -         -         -         -         -         -         -         -
           6         -         -         -         -         -                   -         -         -         -         -         -         -         -
           7         -         -         -         -         -         -                   -         -         -         -         -         -         -
           8         -         -         -         -         -         -         -                   -         -         -         -         -         -
           9         -         -         -         -         -         -         -         -                   -         -         -         -         -
          10         -         -         -         -         -         -         -         -         -                   -         -         -         -
          11         -         -         -         -         -         -         -         -         -         -                   -         -         -
          12         -         -         -         -         -         -         -         -         -         -         -                   -         -
          13         -         -         -         -         -         -         -         -         -         -         -         -                   -
          14         -         -         -         -         -         -         -         -         -         -         -         -         -          


------------------------------------------------------------
Histogram of the number of significant different populations (significance level=0.0500):
------------------------------------------------------------

------------------------------------------------------------------------------------------------------------------------------------------------------------
|         1|         2|         3|         4|         5|         6|         7|         8|         9|        10|        11|        12|        13|        14|
------------------------------------------------------------------------------------------------------------------------------------------------------------
|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|         0|
------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_genot_assignment">&nbsp;</a>

===========================================================
== LIKELIHOOD OF INDIVIDUAL GENOTYPES IN ALL POPULATIONS ==
===========================================================

Reference: <i>Paetkau et al. 1995, 1997</i>
           <i>Waser and Strobeck, 1998.</i>

List of labels for population samples used below:
=================================================

Sample label  	Sample name
------------  	----------
       1:	QBED1-1
       2:	QBED1-2
       3:	QBED2-1
       4:	QBED2-2
       5:	QLL1-1
       6:	QLL1-2
       7:	QLL3-1
       8:	QLL3-2
       9:	QRL1-1
       10:	QRL1-2
       11:	QRL2-1
       12:	QRL2-2
       13:	QTUR-1
       14:	QTUR-2

Sample1
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
QBED1-1-1st	    -9.331   -15.577   -17.425   -12.126   -18.829   -10.838   -15.161   -25.234   -11.717   -12.762   -10.414   -14.822   -15.837   -16.111
 QBED1-1-2	    -6.088    -7.259    -8.900    -7.639    -6.887    -7.671   -11.597   -10.371   -10.598    -6.696    -8.031   -11.386   -20.693    -7.044
 QBED1-1-3	   -10.337   -14.789   -11.851   -13.855   -17.813   -15.286   -19.757   -15.952   -19.040   -17.145   -15.789   -17.353   -19.603   -14.548
 QBED1-1-4	    -9.644   -14.096   -17.033   -14.193   -18.063   -13.339   -19.097   -18.437   -17.855   -13.248   -13.498   -17.518   -25.715   -15.941
QBED1-1-43st	   -16.150   -25.549   -27.830   -26.863   -29.211   -26.610   -27.171   -29.178   -24.007   -25.838   -22.422   -25.381   -23.214   -31.333
QBED1-1-49	    -7.880   -12.592   -14.628   -12.807   -11.952   -11.908   -16.699   -20.370   -14.653   -12.277   -11.733   -13.258   -17.000   -12.279
QBED1-1-50	    -7.385    -9.339   -10.337    -9.343   -12.112    -8.820   -15.261   -14.914   -12.040   -12.856   -11.855    -8.918   -13.365    -9.046
QBED1-1-51	    -7.321   -12.687   -14.574   -12.807    -7.842   -11.741   -16.427   -19.839   -14.376    -9.019   -11.445   -10.000   -20.105   -11.874
QBED1-1-52	    -8.237   -13.786   -14.168   -11.782   -16.269    -9.463   -10.817   -14.923    -9.865    -9.712    -9.486   -13.327   -14.089    -8.497
QBED1-1-6st	   -15.258   -22.497   -21.779   -22.288   -24.538   -21.480   -22.355   -22.229   -31.477   -29.963   -22.761   -23.197   -25.415   -17.953
QBED1-1-61	   -10.876   -10.861   -12.614   -14.059    -8.592   -13.475   -19.157   -15.077   -19.311   -14.123   -10.933   -14.160   -22.322   -13.503
QBED1-1-62	    -4.900    -8.240   -10.516    -7.736    -6.047    -6.959   -10.730   -14.664    -9.292    -8.031    -6.520    -8.961   -13.903    -6.744
QBED1-1-64	   -10.401    -9.783    -9.888    -9.102    -7.849    -9.724    -6.253    -8.069    -6.709    -8.489   -11.269   -10.193    -8.170    -8.502
QBED1-1-N3	    -7.475    -8.550   -10.168    -8.891   -11.952    -8.669   -15.533   -11.781   -13.415   -11.064   -12.055   -10.410   -16.202    -9.164
QBED1-1-N4	    -8.350    -7.365    -9.033    -8.960    -8.170    -7.888   -12.444   -15.445   -14.715    -7.805    -7.092    -8.695   -15.883    -7.238
QBED1-1-N43st	   -12.193   -20.163   -20.951   -20.573   -19.765   -12.953   -21.728   -29.307   -10.253   -19.415   -11.902   -20.704   -20.587   -24.564
QBED1-1-N44st	   -11.143   -13.919   -10.344   -13.459   -13.839   -13.606   -15.939   -20.370   -13.500   -13.266   -14.554   -15.025   -18.581   -14.346
QBED1-1-N51	    -6.713    -8.263   -10.918    -8.342   -12.172    -7.221   -12.794   -15.664   -10.174    -8.137    -7.213   -10.347   -16.372   -12.794
QBED1-1-N52	   -11.302   -10.278   -13.452   -12.301   -19.298   -10.514   -14.113   -16.547   -16.270   -13.874   -10.046   -12.436   -17.581   -20.087
QBED1-1-N6st	   -10.114   -14.061   -14.549    -8.737    -7.676   -12.574   -12.368   -16.236   -14.869    -8.858   -10.304   -14.646   -18.329   -13.707

Sample2
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
QBED1-2-10	    -8.371    -8.176    -9.582    -8.496   -13.051    -7.033   -14.672   -20.901   -10.817    -7.523    -7.846   -10.262   -18.581   -14.464
QBED1-2-12	    -9.980    -9.061   -11.205    -9.025    -9.454   -14.067   -14.220   -11.337   -17.212   -12.257   -11.496   -14.159   -21.406    -9.108
QBED1-2-33	   -15.164    -9.600   -13.868   -10.665    -9.625   -13.693    -8.604    -9.766   -17.092   -12.792   -13.215   -13.286    -8.036   -12.072
QBED1-2-34	   -20.774   -10.971   -14.350   -19.698   -24.079   -18.838   -16.006   -13.349   -22.749   -15.500   -15.006   -19.995   -20.107   -20.020
QBED1-2-36	    -9.603    -7.483    -9.382    -9.566   -13.849    -7.984   -14.236   -15.915   -15.321    -8.316    -7.497    -9.794   -18.486   -12.883
QBED1-2-67	    -5.711    -6.854    -8.728    -8.044    -6.743    -6.422   -12.290   -14.914    -9.070    -7.112    -6.399    -7.820   -16.335    -6.966
QBED1-2-68	   -12.659    -9.339   -13.797   -10.847    -8.793   -12.347    -7.927    -9.333   -10.360   -12.399   -13.620   -12.460    -8.364   -12.311
QBED1-2-69	   -12.060   -11.664   -12.623   -11.788   -19.332   -10.695   -11.628   -14.975   -14.995   -15.720   -11.985   -12.496   -15.878   -15.789
QBED1-2-70	   -28.597   -12.963   -15.834   -23.792   -22.782   -22.728   -17.274   -18.623   -34.266   -16.806   -15.080   -26.326   -24.146   -18.302
QBED1-2-71	   -13.960    -9.562   -15.397   -11.135   -10.185   -13.257   -19.404   -19.707   -20.083   -14.688   -11.235   -15.067   -22.063   -13.576
 QBED1-2-9	   -22.611   -17.441   -19.502   -23.818   -21.047   -22.302   -29.161   -29.234   -28.395   -23.101   -24.638   -21.002   -23.401   -26.558
QBED1-2-N11	   -12.599   -10.113    -9.935   -12.502   -23.815    -9.011   -12.675   -10.309    -8.741   -10.789   -10.013   -10.220   -15.177   -19.014
QBED1-2-N12	   -15.857   -10.756   -14.396   -15.118   -18.666   -13.215   -11.510   -16.706   -12.413   -13.449   -15.412   -14.737   -16.317   -20.207
QBED1-2-N33	   -14.248   -11.044   -15.408   -14.558   -13.268   -14.400   -15.580   -14.952   -17.692   -19.048   -15.020   -19.665   -22.079   -13.829
QBED1-2-N34	   -11.500   -10.129    -8.847    -9.052    -8.502   -10.994    -7.792    -6.974    -8.144    -9.744   -11.886   -11.483   -13.073    -8.461
QBED1-2-N35	   -14.006   -11.479   -12.575   -12.347   -12.045   -13.407   -11.176   -15.161    -9.559   -10.214   -13.442   -13.088   -11.450   -16.050
QBED1-2-N37	   -22.238   -14.101   -15.875   -22.246   -32.012   -17.788   -20.991   -16.633   -37.776   -27.919   -16.065   -24.275   -31.230   -21.604
QBED1-2-N71	   -12.104    -7.857   -11.392   -13.654    -9.985    -9.362   -15.047   -14.553   -13.536    -7.218   -12.615   -11.796   -20.859    -9.758
QBED1-2-N8	   -21.269   -14.909   -22.469   -22.606   -19.940   -20.605   -27.731   -30.901   -26.750   -16.936   -21.747   -20.958   -31.183   -26.808
QBED1-2-N9	   -11.212    -9.754   -10.628   -11.376   -14.072   -13.511   -12.968   -10.963   -14.837   -13.366   -10.876   -10.949   -15.103    -8.625

Sample3
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
QBED2-1-12	   -14.832   -14.828   -13.823   -16.423   -20.864   -13.301   -17.264   -20.390   -15.049   -18.003   -12.656   -16.976   -15.804   -16.368
QBED2-1-13	    -6.648    -6.876    -9.801    -8.044   -11.914    -6.634   -13.574   -12.126   -11.162    -6.291    -7.066   -10.981   -20.559   -12.618
QBED2-1-14	   -36.696   -24.812   -13.181   -34.333   -35.573   -35.859   -19.976   -25.226   -37.635   -28.060   -28.728   -35.634   -38.883   -26.952
QBED2-1-17	   -19.376   -20.622   -13.295   -25.526   -25.958   -25.437   -23.869   -26.871   -25.106   -22.291   -19.217   -25.209   -27.885   -19.074
QBED2-1-18	    -7.539    -9.744   -10.039    -8.697   -12.169    -9.694   -13.875   -13.342   -11.777   -13.133   -12.880   -10.588   -14.058    -8.718
QBED2-1-19	    -9.960    -8.337    -7.759    -8.312    -7.058    -8.733    -7.792    -7.160    -7.021    -8.081    -9.474    -8.428   -10.994    -7.403
QBED2-1-21	   -28.597   -21.272   -13.536   -26.763   -23.475   -23.826   -20.938   -22.980   -31.295   -21.649   -25.379   -22.662   -19.565   -20.579
QBED2-1-22	   -21.897   -16.733   -11.532   -17.392   -17.103   -18.658   -10.401   -15.161   -24.193   -16.318   -18.679   -17.052   -14.988   -15.156
QBED2-1-23	   -15.238   -10.707   -12.157   -11.608   -14.513   -14.394   -15.580    -8.977   -17.365   -18.471   -13.540   -15.276   -15.247   -15.899
QBED2-1-24	   -20.976   -21.234   -11.631   -24.478   -20.583   -23.626   -24.853   -30.000   -23.557   -20.351   -21.378   -19.605   -25.455   -24.631
QBED2-1-26	   -21.621   -22.371   -15.066   -15.756   -16.155   -23.776   -25.014   -23.180   -25.783   -21.679   -24.127   -28.126   -29.436   -21.520
QBED2-1-58	   -21.610   -15.487   -16.098   -21.222   -32.395   -21.483   -18.138   -24.271   -32.320   -28.969   -17.251   -19.630   -24.285   -22.585
QBED2-1-59	   -30.191   -17.209   -11.408   -27.742   -33.409   -27.692   -17.927   -19.851   -30.924   -20.464   -20.698   -28.569   -31.684   -25.428
QBED2-1-N13	   -15.031    -8.920    -8.496   -13.526   -17.946   -13.621   -10.329    -9.357   -12.372   -10.150   -10.921   -15.301   -17.358   -14.951
QBED2-1-N14	   -15.797   -12.850   -10.870   -15.967   -27.317   -12.628   -13.594   -12.295   -10.526   -12.868   -13.921   -13.567   -17.184   -16.149
QBED2-1-N16	   -17.540   -18.678   -12.319   -20.146   -26.121   -17.238   -16.139   -21.921   -20.338   -17.668   -14.338   -16.208   -12.758   -15.665
QBED2-1-N17	   -14.317   -13.100   -10.398   -13.480   -11.639   -12.802   -13.161   -12.964   -12.071   -12.843   -13.138   -12.092   -13.965   -12.571
QBED2-1-N20	   -12.861   -13.514    -9.234   -13.259   -12.741   -12.306   -15.852   -24.095   -13.182   -12.756   -12.896   -13.233   -15.979   -13.869
QBED2-1-N25	   -17.828   -19.562   -14.397   -21.554   -18.488   -21.897   -20.784   -27.598   -20.265   -17.280   -21.513   -18.566   -23.159   -23.584
QBED2-1-N26	   -16.571   -16.155   -12.111   -11.980   -12.254   -17.340   -15.939   -15.198   -16.949   -15.189   -17.978   -20.031   -20.167   -14.994

Sample4
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QBED2-2-1	   -14.427   -12.586   -15.073   -12.599   -16.119   -16.980   -12.522   -14.247   -14.358   -16.065   -13.467   -13.489    -9.962   -20.568
 QBED2-2-2	    -8.727    -7.770    -9.205    -8.555    -8.315    -9.138   -11.751   -10.902   -16.243    -7.389    -8.724   -12.262   -20.241    -7.316
QBED2-2-28	   -13.600   -12.006   -12.232    -9.882   -14.559   -14.136   -15.124   -20.805   -17.456   -14.083   -12.502   -13.872   -17.199   -17.346
QBED2-2-29	   -14.006   -11.806   -11.105   -12.147   -18.477    -9.779   -15.456   -14.885   -13.806   -14.707   -11.403   -15.185   -15.882   -15.975
 QBED2-2-3	   -20.486   -19.570   -16.226   -12.562   -14.280   -20.735   -22.683   -21.351   -26.678   -19.853   -15.876   -17.024   -21.231   -18.638
QBED2-2-31	   -13.896   -13.883   -15.867   -10.971   -14.826   -10.915   -19.830   -23.105   -16.897   -13.492   -14.039   -15.055   -23.134   -11.904
QBED2-2-32	   -13.100   -12.229   -15.983   -11.182   -14.306   -15.706   -13.095    -9.841   -16.651   -18.643   -19.003   -17.125   -12.841   -14.063
QBED2-2-33	   -24.394   -20.203   -24.929   -16.255   -32.063   -21.244   -31.223   -23.180   -30.657   -32.144   -34.227   -29.097   -30.750   -25.479
QBED2-2-34	   -15.966   -13.814   -18.620   -10.894    -9.422    -9.694   -15.580   -19.839   -16.827   -10.855   -13.573   -17.222   -17.029   -13.886
QBED2-2-35	   -18.432   -16.414   -16.137   -12.388   -15.266   -15.534   -15.313   -21.719   -14.584   -14.149   -17.597   -15.264   -14.708   -17.198
QBED2-2-36	   -18.614   -16.327   -17.913   -11.982   -15.084   -16.951   -22.560   -19.832   -24.715   -24.275   -17.396   -19.739   -21.281   -12.995
QBED2-2-37	   -18.918   -19.465   -20.478   -10.442   -16.634   -15.124   -18.809   -19.479   -20.886   -14.074   -14.885   -16.617   -19.380   -21.856
QBED2-2-39	   -15.862   -11.998   -11.404   -11.200   -12.397   -17.996   -12.244   -12.926   -20.415   -16.522   -14.746   -14.487   -15.609   -10.209
 QBED2-2-4	   -12.060   -14.371   -15.356    -8.938    -8.978   -18.636   -12.321    -9.420   -19.386   -14.103   -13.061   -17.051   -19.400   -12.897
QBED2-2-40	   -20.373   -21.457   -20.865   -12.676   -18.936   -20.518   -21.189   -25.522   -21.854   -19.878   -21.241   -22.142   -26.716   -20.385
QBED2-2-56	   -20.085   -24.979   -27.463   -14.334   -18.592   -19.851   -21.130   -24.816   -22.939   -16.602   -16.993   -21.527   -19.421   -23.150
QBED2-2-57	    -9.064    -8.381   -11.066    -9.795   -18.892    -7.741   -16.059   -16.953   -11.363    -8.881    -8.251   -11.568   -20.884   -19.244
QBED2-2-60	   -18.095   -17.493   -18.005   -12.698   -16.293   -16.105   -16.340   -18.380   -27.180   -20.061   -12.542   -12.393   -16.334   -16.516
 QBED2-2-7	   -13.985   -20.056   -20.382   -10.884    -9.543   -19.042   -12.501   -11.943   -18.533   -12.072   -13.482   -18.849   -18.492   -18.534
QBED2-2-N36	   -13.336   -13.131   -13.957    -9.498   -12.558    -9.680   -15.533   -19.173   -13.155   -13.086   -12.771   -11.840   -16.077    -9.890

Sample5
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QLL1-1-10	   -15.724   -11.496   -15.996   -11.541    -8.998    -9.804   -20.794   -20.188   -20.238   -14.825   -11.963   -20.101   -23.903   -15.688
 QLL1-1-11	   -11.767   -13.092   -14.545    -8.891    -9.721   -13.415   -17.207   -12.045   -16.487   -12.500    -9.640   -11.897   -22.707   -12.756
 QLL1-1-12	   -16.080   -15.816   -15.590   -14.511    -9.422   -14.248   -12.874   -19.677   -13.033   -13.628   -15.983   -10.496   -12.239   -15.295
 QLL1-1-13	    -8.168   -13.978   -16.215   -13.346    -8.633   -12.869   -16.612   -20.938   -15.457    -9.766   -12.487   -11.551   -19.448   -12.861
 QLL1-1-14	    -8.456    -9.223   -10.100    -7.803    -5.709    -8.137    -6.417    -8.194    -7.201    -8.083    -8.225    -8.065    -5.825    -7.109
 QLL1-1-15	    -8.168   -13.978   -16.215   -13.346    -8.633   -12.869   -16.612   -20.938   -15.457    -9.766   -12.487   -11.551   -19.448   -12.861
 QLL1-1-16	    -6.627    -6.700    -8.035    -7.505    -5.702    -6.422    -9.651    -9.235    -8.089    -6.301    -6.717    -7.483   -13.502    -6.456
 QLL1-1-1A	   -11.613   -10.948   -12.484   -11.857   -12.235   -12.866   -14.929   -23.973   -16.038   -12.059   -10.151   -12.338   -15.803   -14.230
 QLL1-1-1B	   -13.564   -11.923   -15.154   -16.745   -14.158   -20.959   -23.008   -26.674   -22.872   -16.017   -13.132   -13.077   -23.539   -20.387
 QLL1-1-1C	   -13.239   -13.919   -13.986    -8.784    -9.221   -10.584   -11.494   -16.581   -14.710    -8.270   -14.528   -16.799   -17.365   -10.976
 QLL1-1-1D	    -8.322    -9.098    -8.882    -7.397    -6.634    -8.338    -9.277   -13.342    -9.383    -7.577    -9.190   -10.902   -14.665    -8.268
 QLL1-1-29	   -16.491   -16.623   -19.521   -13.073   -12.575   -21.338   -22.562   -15.147   -33.982   -20.552   -15.156   -24.597   -29.802   -16.553
 QLL1-1-30	   -14.471   -18.122   -18.932    -9.343    -8.603   -14.366   -10.155   -11.656   -16.669    -9.587   -14.661   -17.824   -13.539   -13.940
  QLL1-1-7	   -12.886   -11.128   -12.415    -9.749    -7.444   -14.279   -13.259    -7.754   -17.681   -18.525   -11.808   -14.411   -11.686    -9.226
  QLL1-1-8	   -26.858   -21.140   -20.914   -19.115   -13.780   -20.872   -25.058   -25.042   -24.574   -21.238   -22.283   -20.138   -22.149   -24.703
  QLL1-1-9	   -12.193    -9.742    -9.015    -8.873    -7.569    -9.248    -8.178   -13.430   -13.213    -8.568    -9.263    -8.568    -7.716    -8.644
QLL1-1-N10	   -10.653    -8.492    -8.047    -9.410    -8.100    -9.021    -7.541    -7.788    -6.210    -7.975   -10.860    -8.764   -10.736    -8.319
QLL1-1-N11	   -13.218    -9.031   -10.599   -12.633    -7.626    -9.407    -8.024    -9.175    -8.518    -8.540   -12.506    -8.305    -7.475    -8.102
QLL1-1-N12	   -10.024   -10.233   -11.837   -10.693    -8.100    -9.031   -13.988   -14.319   -12.206   -13.957   -10.391   -13.293   -14.107    -9.688
QLL1-1-N13	   -11.560    -9.878   -14.403    -9.595    -7.403   -12.327    -7.398    -8.579   -11.282   -12.964   -12.294   -12.694    -7.405   -11.283

Sample6
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QLL1-2-2A	    -9.421    -7.328    -8.889    -8.180   -11.766    -7.559   -10.124    -9.418   -13.758    -7.272    -7.183    -9.335   -13.743   -11.568
 QLL1-2-2B	   -16.145   -16.417   -16.773   -13.580   -25.258   -11.627   -13.061   -12.767   -14.014   -12.806   -15.185   -16.971   -15.578   -21.094
 QLL1-2-2C	   -16.733    -9.850   -11.684   -16.034   -12.178   -11.586   -12.039   -16.831   -14.915   -10.173   -14.899    -9.976   -11.508   -10.606
 QLL1-2-2D	   -17.515   -12.381   -16.366   -12.100   -13.309   -11.763   -17.770   -14.975   -17.305   -18.196   -16.741   -18.780   -17.407   -15.498
 QLL1-2-2E	    -8.642    -8.463   -10.016   -10.144   -11.506    -7.585   -10.344   -10.622   -13.977   -12.105    -8.732    -8.687   -15.517    -8.503
 QLL1-2-2F	   -11.293    -9.649   -16.151   -10.375    -8.595    -8.800   -17.151   -17.274   -16.729    -9.235    -9.765   -16.601   -23.007   -13.200
 QLL1-2-2G	   -16.352   -12.424   -15.920   -14.873   -19.953   -10.093   -17.084   -21.950   -20.621   -15.812   -12.818   -16.180   -21.621   -16.106
 QLL1-2-2H	   -13.158   -13.405   -16.217   -14.786   -20.783   -13.590   -23.387   -22.105   -24.727   -24.680   -16.164   -17.615   -20.414   -18.591
 QLL1-2-2I	   -16.758   -20.527   -22.754   -18.131   -25.961   -14.051   -19.124   -29.193   -16.868   -16.602   -14.243   -19.027   -26.129   -20.179
 QLL1-2-2J	   -13.223   -15.231   -15.843   -15.857   -25.038   -10.052   -20.549   -25.890   -15.504   -15.569   -14.806   -16.529   -22.704   -26.140
 QLL1-2-2K	   -14.816   -13.000   -13.894   -15.874   -19.996   -11.434   -10.227   -12.701   -19.055   -20.459   -14.618   -10.767   -13.550   -13.324
 QLL1-2-2L	    -8.569   -10.822   -12.926   -10.267    -9.625    -9.691   -13.353   -21.469   -10.825   -10.686    -9.262   -10.412   -12.744    -9.922
 QLL1-2-2M	    -9.421   -10.146   -12.048    -9.923    -8.111    -8.880   -18.671   -17.845   -17.642   -12.640    -9.899   -16.879   -24.827   -10.959
 QLL1-2-2N	   -10.114    -8.581    -9.177   -10.665   -17.570    -8.252   -10.817   -10.180   -13.892    -8.776    -7.877   -10.028   -14.631   -11.673
 QLL1-2-2P	    -9.149   -10.283    -9.141    -9.795   -11.473    -9.270    -6.232    -6.978    -6.684    -9.811    -9.595    -9.569    -8.755    -6.592
 QLL1-2-2Q	   -10.510    -9.744   -15.626   -10.134    -8.062    -8.258   -16.186   -19.714   -14.661   -10.334    -8.871   -14.417   -19.476   -12.389
QLL1-2-N22	    -8.707    -7.932    -8.258    -8.111    -5.960    -7.434    -7.705    -7.914    -6.703    -7.570    -7.816    -6.636    -8.391    -6.926
QLL1-2-N23	   -11.926   -10.101    -8.929    -9.139    -9.914   -10.922   -11.802   -14.630   -11.253    -8.609   -12.462   -13.797   -19.737   -10.648
QLL1-2-N24	   -13.223   -15.231   -15.843   -15.857   -25.038   -10.052   -20.549   -25.890   -15.504   -15.569   -14.806   -16.529   -22.704   -26.140
QLL1-2-N25	   -20.378   -26.211   -26.298   -22.401   -31.512   -15.143   -28.575   -32.918   -17.387   -25.102   -19.388   -27.246   -29.177   -29.502

Sample7
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
  QLL3-1-1	   -14.722   -13.978   -14.938   -15.359   -14.293   -10.661   -12.311   -20.495   -10.066   -13.674   -11.880    -9.899   -16.512   -11.499
 QLL3-1-10	   -28.419   -24.564   -22.147   -15.381   -17.330   -19.294   -15.402   -26.381   -32.914   -15.048   -25.241   -26.283   -18.083   -18.218
 QLL3-1-11	    -8.796    -8.869    -9.717    -9.498   -11.563    -8.460    -8.958    -9.049   -13.714   -12.383    -9.758   -10.356   -16.210    -8.176
 QLL3-1-14	   -11.436   -10.006    -9.995   -10.104   -12.267    -9.638    -7.284    -8.258   -12.604   -13.246   -11.144    -9.797   -10.966    -9.051
 QLL3-1-16	   -11.095   -10.535   -10.588    -9.303   -12.734   -10.058    -8.717    -8.073    -8.543   -10.204   -10.694   -11.361   -11.857   -13.518
 QLL3-1-17	   -21.922   -16.912   -18.027   -19.391   -29.218   -16.385   -12.153   -15.368   -32.560   -24.661   -17.918   -16.144   -14.202   -20.676
  QLL3-1-2	   -14.673   -16.271   -14.679   -17.844   -22.330   -11.168    -9.564   -14.363    -9.305   -16.148   -11.278   -10.202   -13.935    -9.073
QLL3-1-3st	   -19.952   -16.376   -11.515   -17.574   -16.717   -17.480    -9.996   -15.259   -17.737   -15.520   -19.373   -16.513   -15.182   -15.801
QLL3-1-32st	   -13.872   -22.275   -22.776   -14.467   -18.704   -15.069   -11.695   -21.067   -13.037   -13.750   -15.027   -19.998   -10.430   -13.977
QLL3-1-4st	   -10.940    -8.597    -9.052    -9.279    -7.387    -8.763    -6.135    -8.254   -10.955    -7.924    -9.263    -7.725    -5.772    -7.309
 QLL3-1-46	   -18.363   -16.096   -16.909   -14.913   -11.656   -17.370   -11.474   -11.660   -15.722   -11.590   -17.978   -17.184   -15.688   -15.482
 QLL3-1-47	   -12.481   -12.016   -14.201   -13.091   -11.193   -12.497    -9.786   -15.888    -8.265   -10.463   -12.943   -11.916    -7.507   -15.972
 QLL3-1-48	   -10.093   -10.438    -8.833   -13.081   -17.568    -8.820    -9.092    -9.438    -6.970   -10.578    -9.202    -8.022   -10.166    -7.136
 QLL3-1-52	   -28.196   -16.354   -17.855   -30.748   -33.020   -24.348   -14.873   -18.462   -32.863   -26.676   -21.736   -24.871   -23.842   -19.176
 QLL3-1-54	   -18.337   -12.201   -20.540   -20.814   -22.330   -18.273   -10.817   -13.898   -22.110   -23.476   -19.299   -18.937   -20.105   -18.716
QLL3-1-6st	   -16.373   -13.297   -11.256   -12.098   -15.551   -17.045    -9.836   -13.055   -19.421   -16.316   -14.160   -13.415   -10.144   -10.081
  QLL3-1-7	   -23.547   -13.251   -20.617   -14.737   -15.231   -26.104   -12.716   -10.526   -28.523   -24.843   -22.813   -22.885   -13.760   -19.075
  QLL3-1-9	   -13.041   -12.681    -9.587   -11.382   -14.488   -12.035    -8.804   -12.722    -9.142   -11.554   -13.333   -13.232   -13.738   -10.014
QLL3-1-N10	   -15.813   -12.527   -12.561   -13.197   -12.397   -13.558    -8.235   -11.307   -13.622   -11.156   -15.428   -13.484    -9.539   -11.769
QLL3-1-N11	    -8.456    -9.783    -9.694    -9.595   -10.820    -8.137    -6.417    -8.263    -6.642    -8.894    -8.225    -8.065    -6.019    -6.521

Sample8
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QLL3-2-28	   -18.010   -14.637   -12.837   -16.054   -22.907   -22.577   -13.881    -9.555   -19.964   -23.022   -17.217   -14.784   -14.463   -10.930
 QLL3-2-29	   -17.066   -12.943   -13.145   -11.605   -16.281   -17.753   -11.222    -9.039   -20.526   -16.863   -14.565   -14.721   -12.252   -15.448
 QLL3-2-30	   -22.853   -11.877   -12.367   -21.467   -30.698   -24.222   -16.752   -13.193   -24.465   -19.412   -14.638   -23.108   -24.836   -24.368
 QLL3-2-31	   -14.199   -11.775   -15.961   -13.259   -11.801   -17.863   -12.720    -9.420   -16.927   -18.237   -14.779   -14.486   -11.224   -12.248
 QLL3-2-32	   -34.285   -31.390   -30.475   -18.067   -20.240   -29.147   -33.862   -16.374   -38.077   -35.208   -25.761   -31.143   -32.640   -27.965
 QLL3-2-35	   -11.500   -10.889    -9.952   -11.048   -12.226   -10.043   -11.688    -7.327   -11.795   -14.573   -10.982   -14.411   -14.329    -8.150
 QLL3-2-36	   -22.531   -19.919   -18.987   -17.038   -20.013   -21.181   -16.935   -12.576   -30.586   -23.843   -20.893   -25.541   -25.695   -19.753
 QLL3-2-38	   -14.199   -10.389   -14.708   -13.482   -11.801   -13.794    -7.552   -10.209   -16.927   -13.657   -12.987   -13.569   -10.617   -11.555
 QLL3-2-39	   -13.867   -12.286   -12.197   -11.897   -13.921   -15.666    -8.651    -8.698   -18.348   -15.960   -11.808   -11.137    -6.548    -8.046
 QLL3-2-41	   -31.171   -30.602   -37.934   -27.412   -27.995   -38.345   -33.220   -16.431   -41.020   -41.836   -34.989   -38.440   -31.650   -37.027
 QLL3-2-43	    -9.960    -8.337    -7.759    -8.312    -7.058    -8.733    -7.792    -7.160    -7.021    -8.081    -9.474    -8.428   -10.994    -7.403
 QLL3-2-44	   -29.299   -19.777   -23.439   -22.899   -26.656   -28.991   -18.834   -14.523   -35.194   -31.574   -27.812   -25.087   -21.553   -23.908
 QLL3-2-61	   -13.872   -15.256   -15.535   -17.106   -29.274   -12.968   -14.886   -13.869   -24.051   -25.904   -17.937   -13.234   -14.792   -13.723
 QLL3-2-62	   -16.838   -16.586   -15.772   -16.145   -15.692   -17.293   -13.064   -10.879   -13.188   -15.960   -19.077   -18.693   -18.096   -15.233
QLL3-2-N27st	   -15.797   -11.479   -10.428   -12.384   -12.864   -13.427   -10.944   -10.868    -9.854   -10.448   -15.867   -13.904   -15.844   -12.379
QLL3-2-N29	   -16.758   -17.203   -14.434   -18.131   -21.115   -24.942   -18.018   -11.561   -23.869   -25.482   -19.222   -17.216   -16.084   -14.928
QLL3-2-N30	   -11.079    -8.226   -10.172   -10.104   -18.892    -8.166   -14.893   -12.092   -10.964    -8.304    -9.202   -11.355   -19.960   -19.537
QLL3-2-N32	   -12.465   -13.611   -13.182    -9.005    -8.603   -14.011    -9.769    -9.012   -12.519    -9.233   -11.902   -14.129   -15.218   -12.919
QLL3-2-N35	   -11.500   -10.889    -9.952   -11.048   -12.226   -10.043   -11.688    -7.327   -11.795   -14.573   -10.982   -14.411   -14.329    -8.150
QLL3-2-N38	   -26.003   -21.857   -24.384   -23.228   -25.354   -31.630   -26.585   -16.662   -35.336   -32.534   -31.597   -22.813   -25.565   -20.715

Sample9
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QRL1-1-1A	   -18.091   -13.529   -12.513   -17.649   -17.139   -13.421   -10.257   -16.184   -10.411   -12.141   -18.383   -13.569   -11.870   -12.434
 QRL1-1-1B	   -10.093   -17.765   -18.358   -19.023   -17.568   -10.206   -18.925   -19.271    -8.089   -17.906   -10.589   -17.856   -19.999   -17.472
 QRL1-1-1E	   -12.262   -24.697   -26.344   -22.519   -24.345   -14.952   -21.883   -29.247   -11.031   -21.016   -13.224   -24.798   -20.510   -25.054
 QRL1-1-1F	    -9.960    -7.991    -8.600    -9.210    -7.447    -7.888    -7.726    -9.073    -6.168    -7.059    -9.490    -7.260    -8.000    -8.247
 QRL1-1-1G	   -15.213   -15.387   -17.333   -15.944   -20.763   -11.597   -17.178   -27.114   -13.545   -15.848   -11.622   -12.549   -21.549   -17.726
 QRL1-1-1H	   -12.193   -11.654    -9.576   -13.734   -23.997   -10.483   -10.190    -9.029    -8.005   -12.059   -11.327   -11.483   -14.155   -13.147
 QRL1-1-1J	   -12.753   -12.039   -10.029   -13.821   -19.904   -11.415    -9.635   -10.313    -8.645   -11.431   -12.190   -11.653   -11.978    -9.006
 QRL1-1-1K	    -9.149    -9.436   -10.394    -8.409   -10.942    -8.066    -7.937    -8.133    -7.531    -9.000    -8.343    -8.876    -8.561   -12.348
 QRL1-1-1L	   -10.519    -7.954    -8.660    -8.717   -12.085    -7.696    -9.769    -8.914    -7.585    -7.675    -8.509    -8.022   -10.860   -12.976
 QRL1-1-1M	   -12.732   -10.725   -12.360   -13.399   -11.444   -11.793   -11.074   -15.607    -7.766    -9.949   -12.534   -10.487   -10.073   -15.789
 QRL1-1-1N	   -15.629   -17.072   -17.866   -17.482   -15.833   -16.416   -16.759   -17.760    -9.586   -16.979   -16.591   -17.040   -17.397   -13.004
QRL1-1-1Oh	   -10.093   -10.438    -8.833   -13.081   -17.568    -8.820    -9.092    -9.438    -6.970   -10.578    -9.202    -8.022   -10.166    -7.136
 QRL1-1-1P	   -17.170   -19.056   -20.200   -18.714   -17.317   -18.238   -17.637   -19.552    -8.588   -18.420   -18.325   -19.285   -17.433   -17.655
 QRL1-1-1Q	    -7.896    -9.318   -10.046    -7.803    -5.263    -7.970    -6.145    -7.663    -6.925    -8.489    -7.937    -7.777    -5.959    -6.703
 QRL1-1-1R	   -16.129   -20.437   -22.788   -18.296   -21.168   -18.257   -20.149   -31.001   -13.950   -16.541   -16.451   -20.793   -17.965   -21.795
 QRL1-1-1T	   -11.634   -10.032    -9.921   -11.048    -9.486   -10.270    -7.203    -8.985    -6.203    -9.023   -13.001   -10.364    -9.955    -9.817
 QRL1-1-85	    -7.341    -6.722    -8.638    -7.264   -10.786    -6.260   -10.242    -9.418    -8.389    -6.173    -6.778    -8.747   -14.062   -11.702
 QRL1-1-86	   -24.394   -23.948   -21.037   -21.888   -22.902   -23.602   -22.509   -30.508   -13.228   -22.311   -25.513   -26.324   -27.531   -24.093
QRL1-1-87/1C	   -18.091   -17.193   -15.330   -16.145   -16.223   -17.373   -14.163   -18.960   -10.970   -15.805   -19.770   -19.179   -19.090   -16.791
QRL1-1-88/1D	   -12.193    -9.649   -10.823   -11.454   -14.513    -9.233    -9.179   -10.739    -6.767    -8.617   -12.035    -9.959    -9.822   -15.391

Sample10
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QRL1-2-15	   -10.897   -16.175   -19.019   -13.215   -16.191   -14.663   -16.906   -22.957   -15.896   -11.481   -13.033   -20.696   -23.149   -17.111
 QRL1-2-17	    -9.960    -9.680   -10.788   -13.326   -13.779   -12.154   -15.580   -18.294   -12.903   -10.105    -9.285    -7.948   -16.903   -17.296
 QRL1-2-19	   -11.212    -9.687    -9.347   -11.289   -13.349    -9.154    -9.635   -10.430    -8.082    -8.956    -9.778    -8.598    -9.705    -8.535
 QRL1-2-21	   -11.030   -14.931   -14.527   -13.007    -8.775   -14.081   -14.886   -19.086   -15.272    -9.717   -14.869   -13.204   -20.344   -13.980
 QRL1-2-22	   -14.268   -11.546   -16.679   -11.520   -10.871   -10.058   -19.214   -27.273   -17.353   -10.755   -11.526   -16.864   -21.589   -16.144
 QRL1-2-23	   -18.540   -10.438   -14.864   -20.409   -13.633   -12.979   -17.953   -17.489   -17.043   -10.213   -19.036   -14.687   -23.372   -12.720
 QRL1-2-24	   -13.239   -13.631   -14.833    -9.189   -13.801    -9.381   -13.199   -17.805   -14.998    -8.270   -13.275   -16.106   -17.365   -16.144
 QRL1-2-27	   -16.571   -10.478   -10.915   -14.868   -19.585   -14.511   -14.046   -18.335   -21.336   -10.755   -10.997   -17.675   -20.577   -17.109
 QRL1-2-28	    -7.629    -9.252   -10.045    -8.178    -7.763    -8.713   -12.609   -16.050   -12.156    -7.695    -9.478   -13.135   -21.163    -9.184
 QRL1-2-2A	   -28.675   -36.122   -36.364   -22.041   -27.082   -27.474   -23.497   -27.832   -37.083   -17.502   -30.235   -33.512   -28.547   -31.704
 QRL1-2-2B	   -17.313   -12.607   -13.155   -21.826   -17.305   -16.131   -14.076   -16.309   -13.730   -12.079   -17.293   -10.028   -13.783   -14.836
 QRL1-2-2C	   -13.985   -10.822   -11.714   -13.081   -11.708   -11.116   -10.519   -13.005    -8.406    -9.321   -13.936   -10.657   -10.381   -12.342
 QRL1-2-2D	    -9.421    -7.809    -9.288    -8.267    -7.227    -8.324    -9.297   -10.171   -14.121    -7.049    -7.759    -9.217   -11.700    -7.022
 QRL1-2-2E	   -13.292    -9.002    -8.352   -10.327    -9.527   -10.487    -7.695    -8.319   -11.855    -8.668   -11.554    -9.640   -10.284    -8.591
 QRL1-2-2F	   -16.932   -13.097   -14.126   -13.515   -17.789   -13.851   -13.081   -21.371   -16.081   -11.618   -13.349   -14.039   -10.908   -22.531
 QRL1-2-53	   -12.802   -14.363   -14.622   -10.237   -10.784   -14.871   -13.373   -15.915   -14.470   -10.332   -12.564   -15.446   -18.610   -14.528
 QRL1-2-54	   -14.050   -13.218   -15.803   -15.040   -14.788   -13.886   -20.369   -19.441   -18.169    -9.297   -14.289   -17.134   -26.692   -15.214
 QRL1-2-55	    -7.097    -7.393    -8.258    -7.957    -6.308    -6.741    -9.651   -12.899    -6.990    -7.688    -6.804    -6.279   -10.531    -6.743
 QRL1-2-56	    -8.727    -8.117    -8.835    -7.957   -11.563    -6.866   -11.761   -14.691    -9.560    -7.118    -7.877    -9.638   -15.882   -13.547
 QRL1-2-58	    -8.034    -9.051   -11.733   -10.037    -9.596   -10.762   -14.775   -16.115   -12.590    -8.593    -9.283   -13.871   -22.197   -13.383

Sample11
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QRL2-1-13	   -17.867   -21.293   -23.662   -11.869   -13.186   -20.206   -24.029   -16.427   -24.247   -20.333   -11.731   -13.266   -22.983   -19.959
  QRL2-1-3	   -13.644   -14.702   -16.121   -13.952   -18.507   -10.074   -15.838   -26.519   -13.846   -13.787   -10.614   -13.135   -23.044   -17.093
 QRL2-1-30	    -9.266    -9.467   -10.494   -12.720    -8.546   -12.226   -14.060   -18.355   -12.573    -9.189    -9.168    -7.138   -14.167   -12.056
 QRL2-1-3C	   -10.401   -10.514   -11.540    -9.189    -7.542    -9.691    -8.075   -10.111    -8.864    -9.064    -9.899    -9.739    -7.078    -8.900
 QRL2-1-3D	   -16.838   -10.948   -10.273   -15.474   -14.634   -16.074    -8.081   -10.084   -17.128   -11.785   -13.061   -16.391   -13.655   -10.671
 QRL2-1-3E	   -13.174   -21.750   -20.649   -20.101   -19.319   -12.289   -20.948   -30.629   -11.205   -19.903   -13.482   -21.354   -20.939   -21.753
 QRL2-1-3F	   -10.093   -10.438    -8.833   -13.081   -17.568    -8.820    -9.092    -9.438    -6.970   -10.578    -9.202    -8.022   -10.166    -7.136
  QRL2-1-4	   -11.079   -14.000   -12.752   -14.799   -19.464   -13.556   -20.149   -22.757   -16.921   -13.769   -10.842   -16.031   -20.856   -19.716
 QRL2-1-43	    -7.876   -10.630   -13.626   -10.586    -9.807   -10.535   -14.180   -24.602   -10.462   -10.909    -8.148   -10.529   -12.302   -13.775
 QRL2-1-45	   -15.346   -20.052   -21.642   -17.649   -24.856   -15.939   -21.397   -30.410   -16.975   -17.576   -12.011   -19.707   -22.627   -21.867
 QRL2-1-46	   -12.465   -15.519   -16.025   -13.282   -16.411   -16.617   -15.314   -16.309   -15.755   -11.771   -12.462   -17.866   -22.853   -18.487
 QRL2-1-47	   -18.729   -15.813   -19.174   -18.428   -23.335   -16.169   -19.923   -20.297   -34.499   -27.162   -15.875   -20.909   -22.978   -20.288
 QRL2-1-48	   -17.451   -15.596   -18.993   -17.615   -16.482   -17.446   -25.908   -23.491   -28.916   -19.342   -12.413   -23.071   -30.394   -20.447
 QRL2-1-52	   -10.073   -10.150   -12.826   -12.193   -16.430   -10.620   -17.953   -25.485   -11.263   -11.600    -8.837   -10.892   -18.164   -19.930
  QRL2-1-6	    -8.638    -7.904    -8.764    -8.026    -6.694    -7.782    -8.332   -12.612   -12.053    -8.147    -6.865    -7.032    -8.170    -6.211
  QRL2-1-7	   -15.031   -13.344   -15.856   -15.022   -19.267   -13.995   -19.605   -17.869   -22.592   -13.356    -9.822   -17.717   -26.507   -18.823
 QRL2-1-71	   -17.803   -15.675   -15.764   -16.417   -23.507   -18.702   -15.831   -13.120   -17.874   -19.898   -13.118   -15.553   -15.699   -18.482
 QRL2-1-74	   -12.643   -12.805   -14.721    -9.903   -15.255   -12.261   -19.692   -17.058   -15.564   -13.427    -8.414    -9.429   -20.953   -18.323
  QRL2-1-8	    -8.440    -7.703   -10.229    -9.210    -8.163    -8.987   -12.408   -14.816    -9.664    -7.100    -7.565    -9.563   -14.180   -12.506
  QRL2-1-9	   -21.467   -11.113   -13.567   -16.034   -14.695   -14.905   -14.886   -19.802   -22.938   -12.375   -12.307   -18.940   -18.401   -15.899

Sample12
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QRL2-2-4A	   -27.353   -22.488   -22.440   -27.860   -24.006   -27.055   -25.516   -26.303   -29.549   -22.231   -24.634   -11.314   -25.894   -26.502
 QRL2-2-4B	   -30.547   -29.803   -28.232   -30.815   -36.298   -24.535   -23.654   -32.278   -28.631   -32.921   -26.017   -14.664   -29.283   -25.943
 QRL2-2-4C	   -31.126   -24.876   -28.188   -24.357   -20.856   -26.064   -29.086   -28.732   -27.428   -25.747   -23.182   -14.564   -20.937   -25.301
 QRL2-2-4D	   -10.653   -11.413   -12.372   -14.953   -15.392   -14.731   -21.143   -21.188   -18.205   -14.590   -14.850    -9.899   -18.910   -15.163
 QRL2-2-4E	   -12.801   -12.607   -10.814   -18.691   -20.401   -14.563   -14.702   -15.181   -12.858   -12.718   -11.910    -8.715   -15.845   -12.848
 QRL2-2-4F	   -11.906   -10.265   -11.492   -11.358   -13.637    -9.805    -9.990   -15.222   -18.911   -14.239    -9.861    -9.217   -10.744    -9.357
 QRL2-2-4G	   -16.129   -15.721   -15.849   -10.442   -11.602   -14.553   -17.510   -22.007   -15.683   -14.372   -11.129    -9.808   -13.442   -16.099
 QRL2-2-4H	   -18.432   -21.767   -21.989   -15.811   -17.934   -19.800   -23.166   -26.871   -21.319   -17.195   -16.293   -12.799   -19.948   -26.246
 QRL2-2-4I	    -8.707    -7.932    -8.258    -8.111    -5.960    -7.434    -7.705    -7.914    -6.703    -7.570    -7.816    -6.636    -8.391    -6.926
 QRL2-2-4J	   -25.630   -24.599   -22.763   -17.735   -26.146   -23.595   -21.344   -15.108   -26.166   -24.320   -14.849   -12.164   -13.780   -20.512
 QRL2-2-4K	   -16.154   -13.022   -13.669   -15.479   -25.813   -15.644   -11.782   -11.308   -18.552   -20.611   -15.149   -12.618   -12.443   -16.899
 QRL2-2-4L	   -33.246   -34.093   -34.330   -32.533   -23.674   -32.967   -31.049   -32.023   -32.212   -33.416   -32.712   -15.155   -31.236   -32.189
 QRL2-2-4M	    -8.707    -7.932    -8.258    -8.111    -5.960    -7.434    -7.705    -7.914    -6.703    -7.570    -7.816    -6.636    -8.391    -6.926
 QRL2-2-4P	    -9.842   -10.571   -12.473   -10.606    -8.319   -10.656    -8.379   -13.342    -7.413    -9.587    -9.883    -9.905    -6.512   -13.264
 QRL2-2-4R	   -11.095   -12.075   -11.974   -12.193   -14.181   -12.361    -9.410   -12.722    -8.676   -11.708   -10.848   -12.054   -10.259   -12.930
 QRL2-2-4S	   -10.073    -9.457   -10.187   -10.689   -15.513    -9.521   -15.555   -19.681   -10.282   -10.906    -9.936   -10.892   -19.956   -14.762
 QRL2-2-4V	   -16.956   -15.875   -17.390   -15.898   -15.372   -19.436   -16.853   -17.141   -18.793   -19.407   -17.396   -12.262   -15.450   -15.324
 QRL2-2-4W	   -12.737   -12.578   -14.546   -20.891   -13.883   -19.670   -25.590   -24.816   -24.025   -12.086   -13.807   -12.489   -32.743   -20.182
 QRL2-2-4X	   -17.333   -15.934   -15.939   -15.118   -18.764   -14.344   -14.666   -20.147   -13.639   -14.139   -16.389   -12.288   -14.842   -20.940
 QRL2-2-4Y	   -12.376    -9.262   -10.725   -11.263   -10.984   -10.741   -13.985   -17.362   -21.746    -9.885    -9.171   -10.957   -16.818    -8.897

Sample13
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QTUR-1-1A	   -15.882   -18.168   -15.047   -19.617   -21.355   -18.559   -14.836   -20.378   -19.080   -17.589   -13.977   -16.747    -9.655   -19.353
 QTUR-1-1B	   -13.506   -15.893   -16.830   -12.686   -14.529   -12.448   -10.576   -15.491   -12.715   -11.602   -13.547   -15.179    -9.797   -14.063
 QTUR-1-1C	   -15.452   -12.430   -17.082   -13.091   -11.683   -15.786   -11.423   -20.062   -13.217   -14.601   -15.627   -16.001    -9.645   -20.096
 QTUR-1-1D	   -27.127   -28.922   -31.078   -28.424   -32.332   -32.038   -31.025   -35.099   -40.414   -31.832   -29.623   -29.070   -17.698   -31.372
 QTUR-1-1E	   -21.690   -16.915   -17.731   -15.523   -12.823   -16.646   -14.666   -17.687   -15.431   -14.832   -17.236   -11.189    -9.569   -17.276
 QTUR-1-1F	   -14.273   -10.900   -10.050   -12.275   -14.270   -11.328    -7.927    -9.612   -18.237   -10.920    -9.956    -9.651    -7.158    -7.464
 QTUR-1-1G	   -11.788   -10.427   -11.098    -9.412    -7.830   -10.297    -7.264    -9.418   -13.539    -9.470    -9.611    -9.633    -6.066    -8.074
 QTUR-1-1H	   -14.496   -17.107   -18.284   -12.927   -14.675   -17.427   -10.463   -15.001   -14.953   -16.085   -15.497   -16.160    -8.024   -12.424
 QTUR-1-1I	   -12.481   -11.274   -12.639   -10.288    -8.871   -14.654    -8.517    -9.258   -12.728   -13.945   -12.789   -10.886    -6.414    -9.683
 QTUR-1-1J	   -20.665   -19.552   -14.429   -14.824   -17.180   -17.076   -16.709   -26.992   -22.322   -20.932   -19.161   -17.775   -13.852   -19.674
 QTUR-1-1K	   -14.560   -13.113   -14.548   -10.575    -9.504   -20.428   -10.848    -8.534   -19.601   -20.422   -14.293   -12.747    -7.653   -10.020
 QTUR-1-1L	    -9.282   -11.824   -10.621   -12.773   -16.871    -9.357    -7.531    -9.187    -7.192   -11.497    -9.323    -9.163    -7.733    -6.914
 QTUR-1-1M	   -15.813   -13.228   -14.011   -14.007   -12.705   -13.876   -10.766   -20.999   -14.387   -11.407   -14.042   -12.989    -8.181   -17.397
 QTUR-1-1N	   -10.940    -8.597    -9.052    -9.279    -7.387    -8.763    -6.135    -8.254   -10.955    -7.924    -9.263    -7.725    -5.772    -7.309
 QTUR-1-1O	   -15.035   -11.901   -12.707   -14.403   -15.619   -11.600    -7.963   -11.656   -17.419   -15.009   -14.050   -10.652    -8.915   -11.660
 QTUR-1-1P	   -15.813   -11.844   -12.391   -11.628   -15.182   -12.385    -9.631   -10.581   -20.207   -11.772   -11.115   -12.013    -9.044   -14.278
 QTUR-1-1Q	   -12.534   -14.075   -13.458   -10.951   -13.309   -11.673   -10.010   -13.434    -9.522    -9.815   -12.470   -13.989   -12.470   -10.549
 QTUR-1-1R	   -17.244   -12.740   -12.732   -18.296   -20.661   -16.329   -11.068   -10.441   -14.431   -16.822   -16.377   -11.483    -9.404    -9.278
 QTUR-1-1S	   -18.902   -16.974   -19.530   -18.296   -18.080   -18.863   -14.981   -22.192   -17.527   -16.947   -19.747   -17.024   -12.374   -14.334
 QTUR-1-1T	   -30.117   -17.813   -24.181   -22.432   -17.140   -23.459   -18.336   -15.761   -22.825   -21.004   -25.468   -17.198   -13.183   -22.167

Sample14
----------
                   1         2         3         4         5         6         7         8         9        10        11        12        13        14
 QTUR-2-2A	   -13.649   -11.796   -11.902   -13.687   -15.290   -11.433    -7.896   -10.371   -12.092   -14.827   -15.015   -11.568   -11.970   -11.865
 QTUR-2-2B	   -10.093   -10.438    -8.833   -13.081   -17.568    -8.820    -9.092    -9.438    -6.970   -10.578    -9.202    -8.022   -10.166    -7.136
 QTUR-2-2C	   -19.728   -15.816   -14.900   -18.578   -21.115   -16.803   -15.010   -16.109   -23.869   -16.321   -15.639   -15.383   -14.872   -10.571
 QTUR-2-2D	   -11.921   -12.335   -10.926   -13.689   -18.298   -10.823    -7.685    -9.718   -12.837   -12.190   -10.017   -10.039    -7.281    -7.186
 QTUR-2-2E	    -9.980    -8.802    -8.678    -8.445    -8.690   -10.149   -10.010    -8.826    -9.806    -8.070   -11.075   -12.618   -18.051    -8.397
 QTUR-2-2F	   -16.491   -13.672   -14.143   -14.873   -15.547   -12.623   -20.373   -12.263   -19.912   -20.787   -14.058   -20.810   -19.805   -12.815
 QTUR-2-2G	   -18.095   -22.797   -23.357   -21.678   -22.018   -23.498   -24.896   -23.752   -25.577   -22.047   -23.880   -27.068   -33.238   -14.971
 QTUR-2-2H	   -17.787   -16.080   -16.256   -16.828   -18.410   -17.871   -21.391   -22.984   -18.920   -15.900   -18.742   -20.025   -26.754   -13.551
 QTUR-2-2I	    -7.945    -8.692    -8.710    -7.803    -6.490    -7.089    -9.970   -17.885    -7.855    -7.993    -7.558    -7.335   -10.308    -8.190
 QTUR-2-2J	   -24.726   -16.905   -16.429   -20.214   -17.544   -18.877   -13.014   -18.823   -20.496   -13.345   -22.047   -23.066   -19.185   -15.324
 QTUR-2-2K	   -10.722   -10.255    -9.833   -12.542   -16.875    -8.597    -8.398    -9.369   -12.032   -14.185    -9.831    -7.840   -10.600    -8.385
 QTUR-2-2L	    -9.266    -7.211    -8.340    -8.421    -7.129    -7.888    -9.805    -9.766   -13.734    -6.994    -7.410    -8.359   -13.050    -6.728
 QTUR-2-2M	   -17.649   -15.634   -16.660   -15.022   -14.618   -16.784   -12.629   -18.790   -21.309   -19.918   -12.751   -11.820   -14.696   -10.610
 QTUR-2-2N	    -9.603    -8.463    -9.787    -8.938    -9.269   -13.257   -14.036   -13.903   -15.033   -12.896   -10.542   -11.403   -19.092    -8.408
 QTUR-2-2O	   -16.951   -17.989   -18.461   -13.567   -15.833   -14.139   -16.759   -17.824   -13.396   -17.615   -17.219   -16.704   -17.272   -10.471
 QTUR-2-2P	   -10.130   -11.775   -12.990    -9.189   -11.801   -14.892   -12.720    -8.727   -16.927   -18.237   -14.779   -11.515    -8.253    -9.278
 QTUR-2-2Q	    -4.900    -8.240   -10.516    -7.736    -6.047    -6.959   -10.730   -14.664    -9.292    -8.031    -6.520    -8.961   -13.903    -6.744
 QTUR-2-2R	   -18.784   -14.031   -13.814   -19.751   -22.146   -14.081   -10.442   -13.021   -16.204   -13.682   -16.320   -12.811    -8.834   -10.265
 QTUR-2-2S	   -12.886   -15.721   -13.772   -14.868   -13.982   -15.347   -14.820   -20.147   -19.716   -15.392   -14.278   -14.550   -13.885   -10.632
 QTUR-2-2T	   -13.803   -16.327   -18.024   -12.138   -14.357   -17.427   -12.542   -15.694   -22.519   -16.020   -13.418   -17.258   -13.074   -10.904


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<a name="18_09_11at16_08_26_samp13">&nbsp;</a>

===============================================================================
== Sample : 	QBED1-1
===============================================================================


===============================
== Standard diversity indices : (QBED1-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         9   0.45000   0.43974        34     0.25714
     2        40         5   0.50000   0.61026        32     0.15152
     3        38         8   0.84211   0.75107        20     0.38095
     4        38         4   0.52632   0.42248         6     0.57143
     5        40         4   0.45000   0.49231        13     0.28571
---------------------------------------------------------------------
  Mean    39.200     6.000   0.55368   0.54317    21.000     0.32935
  s.d.     1.095     2.345   0.16456   0.13745    10.770     0.14143
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QBED1-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 31

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.520000 +/-     0.315801

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QBED1-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -65.71016          LnLHood LE :  -72.95444
          Exact P=  0.79960  +-   0.00411 (10100 permutations done)     Chi-square test value=14.48856 (P =  0.99666,  32 d.f.)
Pair(0, 2)
          LnLHood LD :  -80.18860          LnLHood LE :  -93.97229
          Exact P=  0.84574  +-   0.00343 (10100 permutations done)     Chi-square test value=27.56739 (P =  0.99998,  64 d.f.)
Pair(1, 2)
          LnLHood LD :  -80.80250          LnLHood LE :  -93.91005
          Exact P=  0.24802  +-   0.00456 (10100 permutations done)     Chi-square test value=26.21510 (P =  0.75401,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -57.21900          LnLHood LE :  -65.80850
          Exact P=  0.56604  +-   0.00476 (10100 permutations done)     Chi-square test value=17.17900 (P =  0.98491,  32 d.f.)
Pair(1, 3)
          LnLHood LD :  -62.01894          LnLHood LE :  -65.74625
          Exact P=  0.84644  +-   0.00378 (10100 permutations done)     Chi-square test value= 7.45462 (P =  0.96347,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -74.99273          LnLHood LE :  -86.76410
          Exact P=  0.33881  +-   0.00536 (10100 permutations done)     Chi-square test value=23.54275 (P =  0.86049,  32 d.f.)
Pair(0, 4)
          LnLHood LD :  -60.95942          LnLHood LE :  -67.88794
          Exact P=  0.80921  +-   0.00327 (10100 permutations done)     Chi-square test value=13.85704 (P =  0.94981,  24 d.f.)
Pair(1, 4)
          LnLHood LD :  -64.37651          LnLHood LE :  -67.82569
          Exact P=  0.88832  +-   0.00298 (10100 permutations done)     Chi-square test value= 6.89836 (P =  0.86426,  12 d.f.)
Pair(2, 4)
          LnLHood LD :  -78.54843          LnLHood LE :  -88.84354
          Exact P=  0.52238  +-   0.00502 (10100 permutations done)     Chi-square test value=20.59024 (P =  0.66276,  24 d.f.)
Pair(3, 4)
          LnLHood LD :  -51.37009          LnLHood LE :  -60.67975
          Exact P=  0.02594  +-   0.00155 (10100 permutations done)     Chi-square test value=18.61931 (P =  0.09814,  12 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    0    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   + 
       4 |  -   -   -   +   * 

===============================
== Hardy-Weinberg equilibrium : (QBED1-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.45000      0.43974    0.71262  0.00040     1001000
    2      20       0.50000      0.61026    0.10802  0.00030     1001000
    3      19       0.84211      0.75107    0.85711  0.00029     1001000
    4      19       0.52632      0.42248    0.72275  0.00038     1001000
    5      20       0.45000      0.49231    0.27003  0.00043     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp12">&nbsp;</a>

===============================================================================
== Sample : 	QBED1-2
===============================================================================


===============================
== Standard diversity indices : (QBED1-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         3   0.15000   0.23205         4     0.60000
     2        40         5   0.65000   0.62564        32     0.15152
     3        40        10   0.75000   0.85385        36     0.27027
     4        40         4   0.60000   0.66282         6     0.57143
     5        40         6   0.70000   0.73974        16     0.35294
---------------------------------------------------------------------
  Mean    40.000     5.600   0.57000   0.62282    18.800     0.38923
  s.d.     0.000     2.702   0.24135   0.23524    13.121     0.17297
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QBED1-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 37

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.622821 +/-     0.366658

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QBED1-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -47.59446          LnLHood LE :  -53.92923
          Exact P=  0.04564  +-   0.00186 (10100 permutations done)     Chi-square test value=12.66955 (P =  0.12374,  8 d.f.)
Pair(0, 2)
          LnLHood LD :  -81.01986          LnLHood LE :  -86.32369
          Exact P=  0.77228  +-   0.00412 (10100 permutations done)     Chi-square test value=10.60767 (P =  0.91026,  18 d.f.)
Pair(1, 2)
          LnLHood LD :  -94.49554          LnLHood LE : -107.54007
          Exact P=  0.77168  +-   0.00386 (10100 permutations done)     Chi-square test value=26.08906 (P =  0.88800,  36 d.f.)
Pair(0, 3)
          LnLHood LD :  -53.32008          LnLHood LE :  -55.25595
          Exact P=  0.79218  +-   0.00340 (10100 permutations done)     Chi-square test value= 3.87173 (P =  0.69403,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -72.83806          LnLHood LE :  -76.47233
          Exact P=  0.93693  +-   0.00234 (10100 permutations done)     Chi-square test value= 7.26854 (P =  0.83937,  12 d.f.)
Pair(2, 3)
          LnLHood LD :  -92.19053          LnLHood LE : -108.86679
          Exact P=  0.10911  +-   0.00318 (10100 permutations done)     Chi-square test value=33.35252 (P =  0.18568,  27 d.f.)
Pair(0, 4)
          LnLHood LD :  -60.73066          LnLHood LE :  -66.81679
          Exact P=  0.11218  +-   0.00319 (10100 permutations done)     Chi-square test value=12.17227 (P =  0.27369,  10 d.f.)
Pair(1, 4)
          LnLHood LD :  -78.64035          LnLHood LE :  -88.03318
          Exact P=  0.43356  +-   0.00494 (10100 permutations done)     Chi-square test value=18.78565 (P =  0.53580,  20 d.f.)
Pair(2, 4)
          LnLHood LD : -101.97736          LnLHood LE : -120.42763
          Exact P=  0.66000  +-   0.00489 (10100 permutations done)     Chi-square test value=36.90054 (P =  0.79946,  45 d.f.)
Pair(3, 4)
          LnLHood LD :  -80.38555          LnLHood LE :  -89.35989
          Exact P=  0.32614  +-   0.00427 (10100 permutations done)     Chi-square test value=17.94869 (P =  0.26537,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           1    1    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   +   -   -   - 
       1 |  +   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QBED1-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.15000      0.23205    0.10393  0.00030     1001000
    2      20       0.65000      0.62564    0.52352  0.00048     1001000
    3      20       0.75000      0.85385    0.06889  0.00024     1001000
    4      20       0.60000      0.66282    0.61471  0.00049     1001000
    5      20       0.70000      0.73974    0.26146  0.00037     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp11">&nbsp;</a>

===============================================================================
== Sample : 	QBED2-1
===============================================================================


===============================
== Standard diversity indices : (QBED2-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         6   0.30000   0.46795        30     0.19355
     2        40         4   0.55000   0.62949        32     0.12121
     3        40         9   0.45000   0.88077        22     0.39130
     4        40         5   0.50000   0.70256        10     0.45455
     5        40         6   0.70000   0.68974        18     0.31579
---------------------------------------------------------------------
  Mean    40.000     6.000   0.50000   0.67410    22.400     0.29528
  s.d.     0.000     1.871   0.14577   0.14854     8.040     0.12301
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QBED2-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 37

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.674103 +/-     0.391916

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QBED2-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -65.60459          LnLHood LE :  -73.18559
          Exact P=  0.31723  +-   0.00471 (10100 permutations done)     Chi-square test value=15.16200 (P =  0.43981,  15 d.f.)
Pair(0, 2)
          LnLHood LD :  -93.50760          LnLHood LE : -111.41466
          Exact P=  0.05861  +-   0.00227 (10100 permutations done)     Chi-square test value=35.81410 (P =  0.65914,  40 d.f.)
Pair(1, 2)
          LnLHood LD :  -95.37459          LnLHood LE : -112.51894
          Exact P=  0.03436  +-   0.00175 (10100 permutations done)     Chi-square test value=34.28870 (P =  0.07967,  24 d.f.)
Pair(0, 3)
          LnLHood LD :  -73.06665          LnLHood LE :  -82.64150
          Exact P=  0.30564  +-   0.00482 (10100 permutations done)     Chi-square test value=19.14969 (P =  0.51212,  20 d.f.)
Pair(1, 3)
          LnLHood LD :  -74.60606          LnLHood LE :  -83.74578
          Exact P=  0.09782  +-   0.00273 (10100 permutations done)     Chi-square test value=18.27944 (P =  0.10747,  12 d.f.)
Pair(2, 3)
          LnLHood LD : -106.75786          LnLHood LE : -121.97484
          Exact P=  0.58861  +-   0.00499 (10100 permutations done)     Chi-square test value=30.43396 (P =  0.54587,  32 d.f.)
Pair(0, 4)
          LnLHood LD :  -70.50597          LnLHood LE :  -78.02979
          Exact P=  0.77554  +-   0.00459 (10100 permutations done)     Chi-square test value=15.04764 (P =  0.94026,  25 d.f.)
Pair(1, 4)
          LnLHood LD :  -68.50265          LnLHood LE :  -79.13407
          Exact P=  0.03832  +-   0.00240 (10100 permutations done)     Chi-square test value=21.26285 (P =  0.12864,  15 d.f.)
Pair(2, 4)
          LnLHood LD :  -99.34181          LnLHood LE : -117.36313
          Exact P=  0.15069  +-   0.00345 (10100 permutations done)     Chi-square test value=36.04265 (P =  0.64902,  40 d.f.)
Pair(3, 4)
          LnLHood LD :  -75.43372          LnLHood LE :  -88.58997
          Exact P=  0.03000  +-   0.00180 (10100 permutations done)     Chi-square test value=26.31252 (P =  0.15574,  20 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    2    1    1    2
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   +   -   + 
       2 |  -   +   *   -   - 
       3 |  -   -   -   *   + 
       4 |  -   +   -   +   * 

===============================
== Hardy-Weinberg equilibrium : (QBED2-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.30000      0.46795    0.01838  0.00013     1001000
    2      20       0.55000      0.62949    0.34253  0.00050     1001000
    3      20       0.45000      0.88077    0.00000  0.00000     1001000
    4      20       0.50000      0.70256    0.04581  0.00020     1001000
    5      20       0.70000      0.68974    1.00000  0.00000     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp10">&nbsp;</a>

===============================================================================
== Sample : 	QBED2-2
===============================================================================


===============================
== Standard diversity indices : (QBED2-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         7   0.45000   0.50256        24     0.28000
     2        40         6   0.65000   0.76538        32     0.18182
     3        40         8   0.85000   0.80897        16     0.47059
     4        40         5   0.65000   0.67821        10     0.45455
     5        40         5   0.55000   0.63718        13     0.35714
---------------------------------------------------------------------
  Mean    40.000     6.200   0.63000   0.67846    19.000     0.34882
  s.d.     0.000     1.304   0.14832   0.11962     8.000     0.10835
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QBED2-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 38

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.678462 +/-     0.394061

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QBED2-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -78.78010          LnLHood LE :  -89.87660
          Exact P=  0.46653  +-   0.00500 (10100 permutations done)     Chi-square test value=22.19302 (P =  0.84693,  30 d.f.)
Pair(0, 2)
          LnLHood LD :  -82.81399          LnLHood LE :  -95.76443
          Exact P=  0.62683  +-   0.00450 (10100 permutations done)     Chi-square test value=25.90087 (P =  0.97585,  42 d.f.)
Pair(1, 2)
          LnLHood LD :  -88.30984          LnLHood LE : -110.06539
          Exact P=  0.01248  +-   0.00109 (10100 permutations done)     Chi-square test value=43.51110 (P =  0.15312,  35 d.f.)
Pair(0, 3)
          LnLHood LD :  -69.86218          LnLHood LE :  -79.79922
          Exact P=  0.32000  +-   0.00481 (10100 permutations done)     Chi-square test value=19.87408 (P =  0.70391,  24 d.f.)
Pair(1, 3)
          LnLHood LD :  -82.45848          LnLHood LE :  -94.10019
          Exact P=  0.19743  +-   0.00391 (10100 permutations done)     Chi-square test value=23.28342 (P =  0.27509,  20 d.f.)
Pair(2, 3)
          LnLHood LD :  -88.50822          LnLHood LE :  -99.98801
          Exact P=  0.64198  +-   0.00465 (10100 permutations done)     Chi-square test value=22.95958 (P =  0.73506,  28 d.f.)
Pair(0, 4)
          LnLHood LD :  -69.03255          LnLHood LE :  -77.80357
          Exact P=  0.55644  +-   0.00503 (10100 permutations done)     Chi-square test value=17.54205 (P =  0.82464,  24 d.f.)
Pair(1, 4)
          LnLHood LD :  -80.72835          LnLHood LE :  -92.10454
          Exact P=  0.18713  +-   0.00345 (10100 permutations done)     Chi-square test value=22.75238 (P =  0.30112,  20 d.f.)
Pair(2, 4)
          LnLHood LD :  -84.78863          LnLHood LE :  -97.99236
          Exact P=  0.28891  +-   0.00393 (10100 permutations done)     Chi-square test value=26.40746 (P =  0.55066,  28 d.f.)
Pair(3, 4)
          LnLHood LD :  -75.97347          LnLHood LE :  -82.02715
          Exact P=  0.74356  +-   0.00410 (10100 permutations done)     Chi-square test value=12.10737 (P =  0.73656,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    1    1    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   +   -   - 
       2 |  -   +   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QBED2-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.45000      0.50256    0.07341  0.00025     1001000
    2      20       0.65000      0.76538    0.08412  0.00028     1001000
    3      20       0.85000      0.80897    0.25470  0.00043     1001000
    4      20       0.65000      0.67821    0.55751  0.00048     1001000
    5      20       0.55000      0.63718    0.45921  0.00042     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp9">&nbsp;</a>

===============================================================================
== Sample : 	QLL1-1
===============================================================================


===============================
== Standard diversity indices : (QLL1-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         5   0.30000   0.27821        28     0.17241
     2        40         5   0.60000   0.71923        32     0.15152
     3        40         6   0.60000   0.73077        16     0.35294
     4        40         5   0.60000   0.57179        10     0.45455
     5        40         5   0.45000   0.42692        14     0.33333
---------------------------------------------------------------------
  Mean    40.000     5.200   0.51000   0.54538    20.000     0.29295
  s.d.     0.000     0.447   0.13416   0.19410     8.485     0.11478
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QLL1-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 31

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.545385 +/-     0.328389

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QLL1-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -62.20611          LnLHood LE :  -67.00269
          Exact P=  0.81792  +-   0.00381 (10100 permutations done)     Chi-square test value= 9.59317 (P =  0.88699,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -62.86941          LnLHood LE :  -69.99070
          Exact P=  0.44564  +-   0.00494 (10100 permutations done)     Chi-square test value=14.24258 (P =  0.81799,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -87.74997          LnLHood LE :  -95.53849
          Exact P=  0.82802  +-   0.00413 (10100 permutations done)     Chi-square test value=15.57704 (P =  0.74249,  20 d.f.)
Pair(0, 3)
          LnLHood LD :  -48.15536          LnLHood LE :  -56.26576
          Exact P=  0.08990  +-   0.00257 (10100 permutations done)     Chi-square test value=16.22080 (P =  0.43766,  16 d.f.)
Pair(1, 3)
          LnLHood LD :  -73.56268          LnLHood LE :  -81.81355
          Exact P=  0.34990  +-   0.00412 (10100 permutations done)     Chi-square test value=16.50174 (P =  0.41852,  16 d.f.)
Pair(2, 3)
          LnLHood LD :  -77.96985          LnLHood LE :  -84.80156
          Exact P=  0.83218  +-   0.00353 (10100 permutations done)     Chi-square test value=13.66343 (P =  0.84714,  20 d.f.)
Pair(0, 4)
          LnLHood LD :  -43.27981          LnLHood LE :  -49.18860
          Exact P=  0.36455  +-   0.00498 (10100 permutations done)     Chi-square test value=11.81758 (P =  0.75644,  16 d.f.)
Pair(1, 4)
          LnLHood LD :  -69.19312          LnLHood LE :  -74.73640
          Exact P=  0.77010  +-   0.00374 (10100 permutations done)     Chi-square test value=11.08655 (P =  0.80411,  16 d.f.)
Pair(2, 4)
          LnLHood LD :  -70.61923          LnLHood LE :  -77.72440
          Exact P=  0.63733  +-   0.00523 (10100 permutations done)     Chi-square test value=14.21035 (P =  0.81967,  20 d.f.)
Pair(3, 4)
          LnLHood LD :  -56.28967          LnLHood LE :  -63.99946
          Exact P=  0.24772  +-   0.00483 (10100 permutations done)     Chi-square test value=15.41958 (P =  0.49415,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QLL1-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.30000      0.27821    1.00000  0.00000     1001000
    2      20       0.60000      0.71923    0.29323  0.00041     1001000
    3      20       0.60000      0.73077    0.26828  0.00049     1001000
    4      20       0.60000      0.57179    0.33746  0.00040     1001000
    5      20       0.45000      0.42692    0.77772  0.00037     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp8">&nbsp;</a>

===============================================================================
== Sample : 	QLL1-2
===============================================================================


===============================
== Standard diversity indices : (QLL1-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         5   0.25000   0.35128        26     0.18519
     2        40         5   0.55000   0.70256        32     0.15152
     3        38         8   0.78947   0.82788        16     0.47059
     4        40         3   0.40000   0.50897         6     0.42857
     5        40         6   0.50000   0.63846        16     0.35294
---------------------------------------------------------------------
  Mean    39.600     5.400   0.49789   0.60583    19.200     0.31776
  s.d.     0.894     1.817   0.19923   0.18296     8.998     0.12813
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QLL1-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 33

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.589487 +/-     0.350206

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QLL1-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -63.65147          LnLHood LE :  -69.51390
          Exact P=  0.44139  +-   0.00472 (10100 permutations done)     Chi-square test value=11.72486 (P =  0.76269,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -81.61233          LnLHood LE :  -91.03516
          Exact P=  0.55356  +-   0.00513 (10100 permutations done)     Chi-square test value=18.84567 (P =  0.96853,  32 d.f.)
Pair(1, 2)
          LnLHood LD :  -93.73006          LnLHood LE : -110.27165
          Exact P=  0.17960  +-   0.00411 (10100 permutations done)     Chi-square test value=33.08319 (P =  0.41406,  32 d.f.)
Pair(0, 3)
          LnLHood LD :  -48.00909          LnLHood LE :  -52.76551
          Exact P=  0.17772  +-   0.00388 (10100 permutations done)     Chi-square test value= 9.51283 (P =  0.30090,  8 d.f.)
Pair(1, 3)
          LnLHood LD :  -69.71678          LnLHood LE :  -72.00200
          Exact P=  0.90337  +-   0.00295 (10100 permutations done)     Chi-square test value= 4.57044 (P =  0.80235,  8 d.f.)
Pair(2, 3)
          LnLHood LD :  -85.18984          LnLHood LE :  -93.52326
          Exact P=  0.43267  +-   0.00508 (10100 permutations done)     Chi-square test value=16.66685 (P =  0.40747,  16 d.f.)
Pair(0, 4)
          LnLHood LD :  -62.88870          LnLHood LE :  -68.14195
          Exact P=  0.77337  +-   0.00399 (10100 permutations done)     Chi-square test value=10.50650 (P =  0.95803,  20 d.f.)
Pair(1, 4)
          LnLHood LD :  -79.64187          LnLHood LE :  -87.37844
          Exact P=  0.65149  +-   0.00489 (10100 permutations done)     Chi-square test value=15.47314 (P =  0.74872,  20 d.f.)
Pair(2, 4)
          LnLHood LD :  -91.38639          LnLHood LE : -108.89970
          Exact P=  0.18475  +-   0.00406 (10100 permutations done)     Chi-square test value=35.02662 (P =  0.69339,  40 d.f.)
Pair(3, 4)
          LnLHood LD :  -66.35535          LnLHood LE :  -70.63005
          Exact P=  0.60842  +-   0.00500 (10100 permutations done)     Chi-square test value= 8.54942 (P =  0.57533,  10 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QLL1-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.25000      0.35128    0.03346  0.00021     1001000
    2      20       0.55000      0.70256    0.00969  0.00010     1001000
    3      19       0.78947      0.82788    0.35861  0.00035     1001000
    4      20       0.40000      0.50897    0.30392  0.00046     1001000
    5      20       0.50000      0.63846    0.24512  0.00052     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp7">&nbsp;</a>

===============================================================================
== Sample : 	QLL3-1
===============================================================================


===============================
== Standard diversity indices : (QLL3-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         5   0.30000   0.48590        16     0.29412
     2        40         4   0.50000   0.56795        16     0.23529
     3        40         9   0.35000   0.80513        22     0.39130
     4        40         4   0.60000   0.64744         6     0.57143
     5        40         5   0.60000   0.57308        16     0.29412
---------------------------------------------------------------------
  Mean    40.000     5.400   0.47000   0.61590    15.200     0.35725
  s.d.     0.000     2.074   0.13964   0.12025     5.154     0.11822
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QLL3-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 37

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.615897 +/-     0.363244

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QLL3-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -60.73514          LnLHood LE :  -66.12900
          Exact P=  0.40554  +-   0.00496 (10100 permutations done)     Chi-square test value=10.78772 (P =  0.54719,  12 d.f.)
Pair(0, 2)
          LnLHood LD :  -85.60131          LnLHood LE : -100.51538
          Exact P=  0.10188  +-   0.00333 (10100 permutations done)     Chi-square test value=29.82815 (P =  0.57689,  32 d.f.)
Pair(1, 2)
          LnLHood LD :  -85.66425          LnLHood LE :  -99.24316
          Exact P=  0.09059  +-   0.00271 (10100 permutations done)     Chi-square test value=27.15783 (P =  0.29715,  24 d.f.)
Pair(0, 3)
          LnLHood LD :  -68.31460          LnLHood LE :  -71.20620
          Exact P=  0.96881  +-   0.00164 (10100 permutations done)     Chi-square test value= 5.78321 (P =  0.92661,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -64.34607          LnLHood LE :  -69.93398
          Exact P=  0.24525  +-   0.00460 (10100 permutations done)     Chi-square test value=11.17582 (P =  0.26385,  9 d.f.)
Pair(2, 3)
          LnLHood LD :  -89.65084          LnLHood LE : -104.32037
          Exact P=  0.08426  +-   0.00307 (10100 permutations done)     Chi-square test value=29.33906 (P =  0.20763,  24 d.f.)
Pair(0, 4)
          LnLHood LD :  -62.69159          LnLHood LE :  -67.51529
          Exact P=  0.76723  +-   0.00473 (10100 permutations done)     Chi-square test value= 9.64741 (P =  0.88438,  16 d.f.)
Pair(1, 4)
          LnLHood LD :  -58.71874          LnLHood LE :  -66.24307
          Exact P=  0.10673  +-   0.00279 (10100 permutations done)     Chi-square test value=15.04866 (P =  0.23879,  12 d.f.)
Pair(2, 4)
          LnLHood LD :  -91.79859          LnLHood LE : -100.62946
          Exact P=  0.92941  +-   0.00280 (10100 permutations done)     Chi-square test value=17.66173 (P =  0.98107,  32 d.f.)
Pair(3, 4)
          LnLHood LD :  -66.35746          LnLHood LE :  -71.32028
          Exact P=  0.64584  +-   0.00432 (10100 permutations done)     Chi-square test value= 9.92563 (P =  0.62249,  12 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QLL3-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.30000      0.48590    0.04412  0.00022     1001000
    2      20       0.50000      0.56795    0.35542  0.00047     1001000
    3      20       0.35000      0.80513    0.00000  0.00000     1001000
    4      20       0.60000      0.64744    0.04401  0.00021     1001000
    5      20       0.60000      0.57308    1.00000  0.00000     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp6">&nbsp;</a>

===============================================================================
== Sample : 	QLL3-2
===============================================================================


===============================
== Standard diversity indices : (QLL3-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         4   0.20000   0.35256        20     0.19048
     2        40         4   0.35000   0.67051        32     0.12121
     3        40        12   0.70000   0.82308        38     0.30769
     4        38         4   0.52632   0.69986         6     0.57143
     5        40         5   0.45000   0.65897        23     0.20833
---------------------------------------------------------------------
  Mean    39.600     5.800   0.44526   0.64100    23.800     0.27983
  s.d.     0.894     3.493   0.18756   0.17393    10.962     0.15752
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QLL3-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 35

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.627179 +/-     0.368807

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QLL3-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -59.37379          LnLHood LE :  -66.81894
          Exact P=  0.04317  +-   0.00215 (10100 permutations done)     Chi-square test value=14.89030 (P =  0.09399,  9 d.f.)
Pair(0, 2)
          LnLHood LD :  -82.52914          LnLHood LE :  -95.64712
          Exact P=  0.20426  +-   0.00430 (10100 permutations done)     Chi-square test value=26.23597 (P =  0.79211,  33 d.f.)
Pair(1, 2)
          LnLHood LD : -102.51852          LnLHood LE : -112.39001
          Exact P=  0.98832  +-   0.00100 (10100 permutations done)     Chi-square test value=19.74299 (P =  0.96690,  33 d.f.)
Pair(0, 3)
          LnLHood LD :  -67.53875          LnLHood LE :  -73.63958
          Exact P=  0.29861  +-   0.00509 (10100 permutations done)     Chi-square test value=12.20166 (P =  0.42962,  12 d.f.)
Pair(1, 3)
          LnLHood LD :  -80.12568          LnLHood LE :  -90.38247
          Exact P=  0.05881  +-   0.00254 (10100 permutations done)     Chi-square test value=20.51358 (P =  0.05797,  12 d.f.)
Pair(2, 3)
          LnLHood LD :  -98.25389          LnLHood LE : -119.21065
          Exact P=  0.21861  +-   0.00443 (10100 permutations done)     Chi-square test value=41.91353 (P =  0.56143,  44 d.f.)
Pair(0, 4)
          LnLHood LD :  -57.60324          LnLHood LE :  -65.78164
          Exact P=  0.04683  +-   0.00199 (10100 permutations done)     Chi-square test value=16.35680 (P =  0.17543,  12 d.f.)
Pair(1, 4)
          LnLHood LD :  -75.04966          LnLHood LE :  -82.52453
          Exact P=  0.16030  +-   0.00372 (10100 permutations done)     Chi-square test value=14.94976 (P =  0.24420,  12 d.f.)
Pair(2, 4)
          LnLHood LD :  -90.40179          LnLHood LE : -111.35271
          Exact P=  0.02861  +-   0.00152 (10100 permutations done)     Chi-square test value=41.90185 (P =  0.56194,  44 d.f.)
Pair(3, 4)
          LnLHood LD :  -85.61628          LnLHood LE :  -89.34518
          Exact P=  0.98822  +-   0.00102 (10100 permutations done)     Chi-square test value= 7.45780 (P =  0.96340,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           2    1    1    0    2
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   +   -   -   + 
       1 |  +   *   -   -   - 
       2 |  -   -   *   -   + 
       3 |  -   -   -   *   - 
       4 |  +   -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (QLL3-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.20000      0.35256    0.05474  0.00023     1001000
    2      20       0.35000      0.67051    0.00207  0.00005     1001000
    3      20       0.70000      0.82308    0.22615  0.00040     1001000
    4      19       0.52632      0.69986    0.09890  0.00030     1001000
    5      20       0.45000      0.65897    0.02833  0.00017     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp5">&nbsp;</a>

===============================================================================
== Sample : 	QRL1-1
===============================================================================


===============================
== Standard diversity indices : (QRL1-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         3   0.15000   0.14487        12     0.23077
     2        40         3   0.55000   0.59872        16     0.17647
     3        36        10   0.33333   0.85079        38     0.25641
     4        40         2   0.25000   0.40897         2     0.66667
     5        40         5   0.60000   0.64487        16     0.29412
---------------------------------------------------------------------
  Mean    39.200     4.600   0.37667   0.52965    16.800     0.32489
  s.d.     1.789     3.209   0.19315   0.26636    11.771     0.17512
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QRL1-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 32

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.496923 +/-     0.304336

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QRL1-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -40.10794          LnLHood LE :  -40.76189
          Exact P=  0.90564  +-   0.00295 (10100 permutations done)     Chi-square test value= 1.30790 (P =  0.86003,  4 d.f.)
Pair(0, 2)
          LnLHood LD :  -81.75077          LnLHood LE :  -89.46366
          Exact P=  0.09277  +-   0.00286 (10100 permutations done)     Chi-square test value=15.42578 (P =  0.75155,  20 d.f.)
Pair(1, 2)
          LnLHood LD :  -99.91583          LnLHood LE : -109.25459
          Exact P=  0.57020  +-   0.00437 (10100 permutations done)     Chi-square test value=18.67753 (P =  0.54287,  20 d.f.)
Pair(0, 3)
          LnLHood LD :  -28.93325          LnLHood LE :  -30.54649
          Exact P=  0.19436  +-   0.00402 (10100 permutations done)     Chi-square test value= 3.22648 (P =  0.19924,  2 d.f.)
Pair(1, 3)
          LnLHood LD :  -48.95513          LnLHood LE :  -50.33743
          Exact P=  0.33089  +-   0.00452 (10100 permutations done)     Chi-square test value= 2.76460 (P =  0.25100,  2 d.f.)
Pair(2, 3)
          LnLHood LD :  -90.55420          LnLHood LE :  -99.03919
          Exact P=  0.10228  +-   0.00287 (10100 permutations done)     Chi-square test value=16.96998 (P =  0.07503,  10 d.f.)
Pair(0, 4)
          LnLHood LD :  -45.16546          LnLHood LE :  -51.03803
          Exact P=  0.03653  +-   0.00171 (10100 permutations done)     Chi-square test value=11.74514 (P =  0.16294,  8 d.f.)
Pair(1, 4)
          LnLHood LD :  -61.49436          LnLHood LE :  -70.82897
          Exact P=  0.00970  +-   0.00096 (10100 permutations done)     Chi-square test value=18.66922 (P =  0.01673,  8 d.f.)
Pair(2, 4)
          LnLHood LD : -104.04325          LnLHood LE : -119.53073
          Exact P=  0.61109  +-   0.00545 (10100 permutations done)     Chi-square test value=30.97495 (P =  0.84630,  40 d.f.)
Pair(3, 4)
          LnLHood LD :  -59.17334          LnLHood LE :  -60.61357
          Exact P=  0.68347  +-   0.00417 (10100 permutations done)     Chi-square test value= 2.88046 (P =  0.57803,  4 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           1    1    0    0    2
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   + 
       1 |  -   *   -   -   + 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  +   +   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QRL1-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.15000      0.14487    1.00000  0.00000     1001000
    2      20       0.55000      0.59872    0.17219  0.00037     1001000
    3      18       0.33333      0.85079    0.00000  0.00000     1001000
    4      20       0.25000      0.40897    0.10690  0.00031     1001000
    5      20       0.60000      0.64487    0.26108  0.00035     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp4">&nbsp;</a>

===============================================================================
== Sample : 	QRL1-2
===============================================================================


===============================
== Standard diversity indices : (QRL1-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         4   0.25000   0.27692        14     0.26667
     2        40         4   0.65000   0.66795        16     0.23529
     3        40         7   0.65000   0.77436        14     0.46667
     4        40         3   0.35000   0.60897         6     0.42857
     5        40         7   0.75000   0.71026        20     0.33333
---------------------------------------------------------------------
  Mean    40.000     5.000   0.53000   0.60769    14.000     0.34611
  s.d.     0.000     1.871   0.21679   0.19451     4.561     0.08954
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QRL1-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 34

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.607692 +/-     0.359195

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QRL1-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -53.91255          LnLHood LE :  -57.80768
          Exact P=  0.52980  +-   0.00512 (10100 permutations done)     Chi-square test value= 7.79025 (P =  0.55541,  9 d.f.)
Pair(0, 2)
          LnLHood LD :  -68.55866          LnLHood LE :  -76.16763
          Exact P=  0.31634  +-   0.00545 (10100 permutations done)     Chi-square test value=15.21794 (P =  0.64695,  18 d.f.)
Pair(1, 2)
          LnLHood LD :  -85.27865          LnLHood LE :  -93.90670
          Exact P=  0.60842  +-   0.00484 (10100 permutations done)     Chi-square test value=17.25611 (P =  0.50557,  18 d.f.)
Pair(0, 3)
          LnLHood LD :  -50.99772          LnLHood LE :  -54.15486
          Exact P=  0.50436  +-   0.00497 (10100 permutations done)     Chi-square test value= 6.31428 (P =  0.38892,  6 d.f.)
Pair(1, 3)
          LnLHood LD :  -70.03833          LnLHood LE :  -71.89394
          Exact P=  0.86396  +-   0.00328 (10100 permutations done)     Chi-square test value= 3.71122 (P =  0.71569,  6 d.f.)
Pair(2, 3)
          LnLHood LD :  -84.69326          LnLHood LE :  -90.25389
          Exact P=  0.71624  +-   0.00424 (10100 permutations done)     Chi-square test value=11.12126 (P =  0.51856,  12 d.f.)
Pair(0, 4)
          LnLHood LD :  -64.52518          LnLHood LE :  -69.18484
          Exact P=  0.84139  +-   0.00359 (10100 permutations done)     Chi-square test value= 9.31931 (P =  0.95188,  18 d.f.)
Pair(1, 4)
          LnLHood LD :  -74.58967          LnLHood LE :  -86.92392
          Exact P=  0.06505  +-   0.00215 (10100 permutations done)     Chi-square test value=24.66849 (P =  0.13432,  18 d.f.)
Pair(2, 4)
          LnLHood LD :  -91.88712          LnLHood LE : -105.28386
          Exact P=  0.77208  +-   0.00441 (10100 permutations done)     Chi-square test value=26.79350 (P =  0.86733,  36 d.f.)
Pair(3, 4)
          LnLHood LD :  -75.73087          LnLHood LE :  -83.27110
          Exact P=  0.30891  +-   0.00455 (10100 permutations done)     Chi-square test value=15.08045 (P =  0.23707,  12 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QRL1-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.25000      0.27692    0.35273  0.00042     1001000
    2      20       0.65000      0.66795    0.05007  0.00020     1001000
    3      20       0.65000      0.77436    0.04781  0.00024     1001000
    4      20       0.35000      0.60897    0.01557  0.00012     1001000
    5      20       0.75000      0.71026    0.26795  0.00041     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp2">&nbsp;</a>

===============================================================================
== Sample : 	QRL2-1
===============================================================================


===============================
== Standard diversity indices : (QRL2-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         8   0.30000   0.43718        16     0.47059
     2        40         5   0.55000   0.66282        32     0.15152
     3        38         6   0.57895   0.77809        14     0.40000
     4        40         4   0.55000   0.58077         6     0.57143
     5        40         7   0.70000   0.68205        40     0.17073
---------------------------------------------------------------------
  Mean    39.600     6.000   0.53579   0.62818    21.600     0.35285
  s.d.     0.894     1.581   0.14563   0.12777    12.484     0.16587
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QRL2-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 35

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.612821 +/-     0.361726

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QRL2-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -67.16801          LnLHood LE :  -77.15128
          Exact P=  0.39446  +-   0.00448 (10100 permutations done)     Chi-square test value=19.96654 (P =  0.86568,  28 d.f.)
Pair(0, 2)
          LnLHood LD :  -85.10707          LnLHood LE :  -95.45978
          Exact P=  0.85119  +-   0.00339 (10100 permutations done)     Chi-square test value=20.70542 (P =  0.99763,  42 d.f.)
Pair(1, 2)
          LnLHood LD :  -85.72009          LnLHood LE :  -99.25457
          Exact P=  0.09386  +-   0.00303 (10100 permutations done)     Chi-square test value=27.06897 (P =  0.30125,  24 d.f.)
Pair(0, 3)
          LnLHood LD :  -61.04827          LnLHood LE :  -69.80993
          Exact P=  0.35386  +-   0.00456 (10100 permutations done)     Chi-square test value=17.52333 (P =  0.67894,  21 d.f.)
Pair(1, 3)
          LnLHood LD :  -66.10289          LnLHood LE :  -73.60473
          Exact P=  0.14366  +-   0.00337 (10100 permutations done)     Chi-square test value=15.00367 (P =  0.24124,  12 d.f.)
Pair(2, 3)
          LnLHood LD :  -79.68527          LnLHood LE :  -91.91322
          Exact P=  0.04752  +-   0.00209 (10100 permutations done)     Chi-square test value=24.45591 (P =  0.14065,  18 d.f.)
Pair(0, 4)
          LnLHood LD :  -74.51740          LnLHood LE :  -83.44987
          Exact P=  0.90911  +-   0.00310 (10100 permutations done)     Chi-square test value=17.86494 (P =  0.99960,  42 d.f.)
Pair(1, 4)
          LnLHood LD :  -75.24717          LnLHood LE :  -87.24467
          Exact P=  0.16238  +-   0.00375 (10100 permutations done)     Chi-square test value=23.99501 (P =  0.46188,  24 d.f.)
Pair(2, 4)
          LnLHood LD :  -90.64861          LnLHood LE : -105.55316
          Exact P=  0.38366  +-   0.00485 (10100 permutations done)     Chi-square test value=29.80911 (P =  0.75689,  36 d.f.)
Pair(3, 4)
          LnLHood LD :  -71.78706          LnLHood LE :  -79.90332
          Exact P=  0.41921  +-   0.00496 (10100 permutations done)     Chi-square test value=16.23251 (P =  0.57632,  18 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    1    1    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   +   - 
       3 |  -   -   +   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QRL2-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.30000      0.43718    0.14827  0.00028     1001000
    2      20       0.55000      0.66282    0.57083  0.00039     1001000
    3      19       0.57895      0.77809    0.16831  0.00042     1001000
    4      20       0.55000      0.58077    0.77622  0.00036     1001000
    5      20       0.70000      0.68205    0.42841  0.00053     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp3">&nbsp;</a>

===============================================================================
== Sample : 	QRL2-2
===============================================================================


===============================
== Standard diversity indices : (QRL2-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         9   0.50000   0.71923        36     0.24324
     2        40         3   0.45000   0.56795        16     0.17647
     3        40         6   0.60000   0.78205        14     0.40000
     4        40         4   0.40000   0.58590         6     0.57143
     5        34         5   0.35294   0.49198        16     0.29412
---------------------------------------------------------------------
  Mean    38.800     5.400   0.46059   0.62942    17.600     0.33705
  s.d.     2.683     2.302   0.09534   0.11824     9.912     0.13808
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QRL2-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 35

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.601795 +/-     0.356284

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QRL2-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -74.62055          LnLHood LE :  -87.54015
          Exact P=  0.01574  +-   0.00113 (10100 permutations done)     Chi-square test value=25.83920 (P =  0.05633,  16 d.f.)
Pair(0, 2)
          LnLHood LD :  -93.63250          LnLHood LE : -111.52954
          Exact P=  0.23436  +-   0.00427 (10100 permutations done)     Chi-square test value=35.79410 (P =  0.66002,  40 d.f.)
Pair(1, 2)
          LnLHood LD :  -79.47735          LnLHood LE :  -84.61133
          Exact P=  0.56772  +-   0.00442 (10100 permutations done)     Chi-square test value=10.26797 (P =  0.41731,  10 d.f.)
Pair(0, 3)
          LnLHood LD :  -84.65304          LnLHood LE :  -93.75739
          Exact P=  0.69703  +-   0.00471 (10100 permutations done)     Chi-square test value=18.20870 (P =  0.79277,  24 d.f.)
Pair(1, 3)
          LnLHood LD :  -63.18058          LnLHood LE :  -66.83917
          Exact P=  0.36663  +-   0.00547 (10100 permutations done)     Chi-square test value= 7.31718 (P =  0.29251,  6 d.f.)
Pair(2, 3)
          LnLHood LD :  -84.39237          LnLHood LE :  -90.82856
          Exact P=  0.73228  +-   0.00473 (10100 permutations done)     Chi-square test value=12.87240 (P =  0.61215,  15 d.f.)
Pair(0, 4)
          LnLHood LD :  -84.88443          LnLHood LE : -103.89757
          Exact P=  0.05277  +-   0.00231 (10100 permutations done)     Chi-square test value=38.02628 (P =  0.55941,  40 d.f.)
Pair(1, 4)
          LnLHood LD :  -72.31619          LnLHood LE :  -76.97935
          Exact P=  0.64495  +-   0.00452 (10100 permutations done)     Chi-square test value= 9.32632 (P =  0.50144,  10 d.f.)
Pair(2, 4)
          LnLHood LD :  -89.54727          LnLHood LE : -100.96875
          Exact P=  0.49406  +-   0.00427 (10100 permutations done)     Chi-square test value=22.84295 (P =  0.58673,  25 d.f.)
Pair(3, 4)
          LnLHood LD :  -76.08260          LnLHood LE :  -83.19659
          Exact P=  0.47475  +-   0.00514 (10100 permutations done)     Chi-square test value=14.22798 (P =  0.50831,  15 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           1    1    0    0    0
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   +   -   -   - 
       1 |  +   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  -   -   -   *   - 
       4 |  -   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QRL2-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.50000      0.71923    0.02707  0.00015     1001000
    2      20       0.45000      0.56795    0.49638  0.00051     1001000
    3      20       0.60000      0.78205    0.14759  0.00031     1001000
    4      20       0.40000      0.58590    0.01681  0.00013     1001000
    5      17       0.35294      0.49198    0.01623  0.00011     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp1">&nbsp;</a>

===============================================================================
== Sample : 	QTUR-1
===============================================================================


===============================
== Standard diversity indices : (QTUR-1)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         7   0.45000   0.43590        34     0.20000
     2        40         3   0.45000   0.50128        16     0.17647
     3        40         7   0.55000   0.72692        16     0.41176
     4        40         5   0.65000   0.75769        10     0.45455
     5        40         5   0.45000   0.51795        16     0.29412
---------------------------------------------------------------------
  Mean    40.000     5.400   0.51000   0.58795    18.400     0.30738
  s.d.     0.000     1.673   0.08944   0.14462     8.139     0.11081
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QTUR-1)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 34

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.587949 +/-     0.349446

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QTUR-1)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -55.41750          LnLHood LE :  -60.85904
          Exact P=  0.48614  +-   0.00453 (10100 permutations done)     Chi-square test value=10.88308 (P =  0.53896,  12 d.f.)
Pair(0, 2)
          LnLHood LD :  -72.59490          LnLHood LE :  -83.59393
          Exact P=  0.58832  +-   0.00494 (10100 permutations done)     Chi-square test value=21.99804 (P =  0.96783,  36 d.f.)
Pair(1, 2)
          LnLHood LD :  -74.23698          LnLHood LE :  -78.02796
          Exact P=  0.92257  +-   0.00236 (10100 permutations done)     Chi-square test value= 7.58195 (P =  0.81689,  12 d.f.)
Pair(0, 3)
          LnLHood LD :  -68.55442          LnLHood LE :  -81.04703
          Exact P=  0.03634  +-   0.00171 (10100 permutations done)     Chi-square test value=24.98522 (P =  0.40656,  24 d.f.)
Pair(1, 3)
          LnLHood LD :  -70.49860          LnLHood LE :  -75.48106
          Exact P=  0.36455  +-   0.00485 (10100 permutations done)     Chi-square test value= 9.96493 (P =  0.26750,  8 d.f.)
Pair(2, 3)
          LnLHood LD :  -85.49205          LnLHood LE :  -98.21594
          Exact P=  0.15792  +-   0.00317 (10100 permutations done)     Chi-square test value=25.44779 (P =  0.38176,  24 d.f.)
Pair(0, 4)
          LnLHood LD :  -54.03376          LnLHood LE :  -66.74402
          Exact P=  0.03891  +-   0.00178 (10100 permutations done)     Chi-square test value=25.42051 (P =  0.38321,  24 d.f.)
Pair(1, 4)
          LnLHood LD :  -56.67776          LnLHood LE :  -61.17805
          Exact P=  0.30406  +-   0.00437 (10100 permutations done)     Chi-square test value= 9.00058 (P =  0.34225,  8 d.f.)
Pair(2, 4)
          LnLHood LD :  -74.04092          LnLHood LE :  -83.91293
          Exact P=  0.34000  +-   0.00480 (10100 permutations done)     Chi-square test value=19.74401 (P =  0.71124,  24 d.f.)
Pair(3, 4)
          LnLHood LD :  -73.79169          LnLHood LE :  -81.36604
          Exact P=  0.35366  +-   0.00487 (10100 permutations done)     Chi-square test value=15.14869 (P =  0.51378,  16 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           2    0    0    1    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   +   + 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   - 
       3 |  +   -   -   *   - 
       4 |  +   -   -   -   * 

===============================
== Hardy-Weinberg equilibrium : (QTUR-1)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.45000      0.43590    0.37117  0.00025     1001000
    2      20       0.45000      0.50128    0.39153  0.00052     1001000
    3      20       0.55000      0.72692    0.25280  0.00033     1001000
    4      20       0.65000      0.75769    0.13457  0.00033     1001000
    5      20       0.45000      0.51795    0.61951  0.00040     1001000
------------------------------------------------------------------------


<a name="18_09_11at16_08_26_samp0">&nbsp;</a>

===============================================================================
== Sample : 	QTUR-2
===============================================================================


===============================
== Standard diversity indices : (QTUR-2)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 40
No. of loci              : 5
No. of usable loci       : 5 loci with less than 30.00 % missing data
No. of polymorphic loci  : 5


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1        40         8   0.40000   0.40128        38     0.20513
     2        40         4   0.30000   0.60385        32     0.12121
     3        40        10   0.75000   0.83974        42     0.23256
     4        40         4   0.45000   0.66026         6     0.57143
     5        40         3   0.20000   0.43205         6     0.42857
---------------------------------------------------------------------
  Mean    40.000     5.800   0.42000   0.58744    24.800     0.31178
  s.d.     0.000     3.033   0.20797   0.17891    15.677     0.16433
---------------------------------------------------------------------



================================
== Molecular diversity indices : (QTUR-2)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 40.00000
No. of haplotypes                   : 36

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 5


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.587436 +/-     0.349192

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (QTUR-2)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD :  -65.33204          LnLHood LE :  -69.29855
          Exact P=  0.98465  +-   0.00114 (10100 permutations done)     Chi-square test value= 7.93301 (P =  0.99542,  21 d.f.)
Pair(0, 2)
          LnLHood LD :  -83.63364          LnLHood LE :  -98.36852
          Exact P=  0.52188  +-   0.00454 (10100 permutations done)     Chi-square test value=29.46976 (P =  0.99990,  63 d.f.)
Pair(1, 2)
          LnLHood LD :  -94.65233          LnLHood LE : -102.09945
          Exact P=  0.94089  +-   0.00256 (10100 permutations done)     Chi-square test value=14.89426 (P =  0.97089,  27 d.f.)
Pair(0, 3)
          LnLHood LD :  -66.03439          LnLHood LE :  -74.36361
          Exact P=  0.52713  +-   0.00550 (10100 permutations done)     Chi-square test value=16.65844 (P =  0.73159,  21 d.f.)
Pair(1, 3)
          LnLHood LD :  -72.96367          LnLHood LE :  -78.09454
          Exact P=  0.38436  +-   0.00482 (10100 permutations done)     Chi-square test value=10.26173 (P =  0.32972,  9 d.f.)
Pair(2, 3)
          LnLHood LD :  -96.19119          LnLHood LE : -107.16452
          Exact P=  0.81337  +-   0.00442 (10100 permutations done)     Chi-square test value=21.94665 (P =  0.74008,  27 d.f.)
Pair(0, 4)
          LnLHood LD :  -52.59489          LnLHood LE :  -58.75350
          Exact P=  0.34624  +-   0.00409 (10100 permutations done)     Chi-square test value=12.31722 (P =  0.58084,  14 d.f.)
Pair(1, 4)
          LnLHood LD :  -59.68507          LnLHood LE :  -62.48443
          Exact P=  0.44386  +-   0.00526 (10100 permutations done)     Chi-square test value= 5.59872 (P =  0.46961,  6 d.f.)
Pair(2, 4)
          LnLHood LD :  -79.84586          LnLHood LE :  -91.55441
          Exact P=  0.04683  +-   0.00195 (10100 permutations done)     Chi-square test value=23.41710 (P =  0.17507,  18 d.f.)
Pair(3, 4)
          LnLHood LD :  -63.46082          LnLHood LE :  -67.54950
          Exact P=  0.25010  +-   0.00386 (10100 permutations done)     Chi-square test value= 8.17736 (P =  0.22540,  6 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4
---------------------------------------------------------------------
           0    0    1    0    1
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          --------------------
Locus  # |  0|  1|  2|  3|  4|
          --------------------
       0 |  *   -   -   -   - 
       1 |  -   *   -   -   - 
       2 |  -   -   *   -   + 
       3 |  -   -   -   *   - 
       4 |  -   -   +   -   * 

===============================
== Hardy-Weinberg equilibrium : (QTUR-2)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1      20       0.40000      0.40128    0.62605  0.00031     1001000
    2      20       0.30000      0.60385    0.00691  0.00008     1001000
    3      20       0.75000      0.83974    0.00663  0.00009     1001000
    4      20       0.45000      0.66026    0.03839  0.00019     1001000
    5      20       0.20000      0.43205    0.00956  0.00010     1001000
------------------------------------------------------------------------




================================================================================
== Summary of computations done within populations
================================================================================

<a name="18_09_11at16_08_26_comp_sum_Basic">&nbsp;</a>

----------------
Basic properties
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
          Statistics     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d.
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  No. of gene copies          40          40          40          40          40          40          40          40          40          40          40          40          40          40      40.000       0.000
         No. of loci           5           5           5           5           5           5           5           5           5           5           5           5           5           5       5.000       0.000
  No. of usable loci           5           5           5           5           5           5           5           5           5           5           5           5           5           5       5.000       0.000
  No. of polym. loci           5           5           5           5           5           5           5           5           5           5           5           5           5           5       5.000       0.000
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_comp_sum_het">&nbsp;</a>

-----------------------
Expected heterozygosity
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d.   Tot. Het.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1     0.43974     0.23205     0.46795     0.50256     0.27821     0.35128     0.48590     0.35256     0.14487     0.27692     0.43718     0.71923     0.43590     0.40128     0.39469     0.14038     0.40447
     2     0.61026     0.62564     0.62949     0.76538     0.71923     0.70256     0.56795     0.67051     0.59872     0.66795     0.66282     0.56795     0.50128     0.60385     0.63526     0.06888     0.65234
     3     0.75107     0.85385     0.88077     0.80897     0.73077     0.82788     0.80513     0.82308     0.85079     0.77436     0.77809     0.78205     0.72692     0.83974     0.80239     0.04715     0.83779
     4     0.42248     0.66282     0.70256     0.67821     0.57179     0.50897     0.64744     0.69986     0.40897     0.60897     0.58077     0.58590     0.75769     0.66026     0.60691     0.10317     0.61788
     5     0.49231     0.73974     0.68974     0.63718     0.42692     0.63846     0.57308     0.65897     0.64487     0.71026     0.68205     0.49198     0.51795     0.43205     0.59540     0.10503     0.61735
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean     0.54317     0.62282     0.67410     0.67846     0.54538     0.60583     0.61590     0.64100     0.52965     0.60769     0.62818     0.62942     0.58795     0.58744     0.60693     0.04528     0.62597
  s.d.     0.13745     0.23524     0.14854     0.11962     0.19410     0.18296     0.12025     0.17393     0.26636     0.19451     0.12777     0.11824     0.14462     0.17891     0.16732     0.04517     0.13767
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_comp_sum_thetaH">&nbsp;</a>

------------------------------------------
Theta(H) under the stepwise mutation model
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1     1.52947     2.30578     1.50825     1.50005     1.98995     1.69411     1.50159     1.69046     3.53604     1.99704     1.53208     1.97601     1.53342     1.58112     1.84824     0.52741
     2     1.60222     1.63480     1.64378     2.28441     1.97601     1.89271     1.53763     1.76321     1.58112     1.75435     1.73724     1.53763     1.50001     1.59016     1.71681     0.20618
     3     2.17429     3.50664     4.26123     2.73552     2.04135     3.00891     2.68681     2.93356     3.43875     2.36160     2.39580     2.43346     2.01882     3.21542     2.80087     0.61880
     4     1.54927     1.73724     1.89271     1.79102     1.54210     1.50064     1.69046     1.88031     1.56856     1.59974     1.55359     1.56082     2.22339     1.72898     1.70135     0.19081
     5     1.50047     2.09709     1.83648     1.66280     1.54365     1.66611     1.54365     1.72492     1.68329     1.92963     1.80566     1.50051     1.50258     1.53763     1.68104     0.17575
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean     1.51502     1.62843     1.77595     1.79199     1.51661     1.59381     1.61356     1.67278     1.50706     1.59729     1.64069     1.64362     1.56386     1.56309     1.61598     0.08754
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_comp_sum_numAll">&nbsp;</a>

-----------------
Number of alleles
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d. Tot. number
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1           9           3           6           7           5           5           5           4           3           4           8           9           7           8       5.929       2.093          20
     2           5           5           4           6           5           5           4           4           3           4           5           3           3           4       4.286       0.914           6
     3           8          10           9           8           6           8           9          12          10           7           6           6           7          10       8.286       1.816          21
     4           4           4           5           5           5           3           4           4           2           3           4           4           5           4       4.000       0.877           5
     5           4           6           6           5           5           6           5           5           5           7           7           5           5           3       5.286       1.069          11
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean       6.000       5.600       6.000       6.200       5.200       5.400       5.400       5.800       4.600       5.000       6.000       5.400       5.400       5.800       5.557       0.445      12.600
  s.d.       2.345       2.702       1.871       1.304       0.447       1.817       2.074       3.493       3.209       1.871       1.581       2.302       1.673       3.033       2.123       0.809       6.771
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_comp_sum_allRange">&nbsp;</a>

------------------
Allelic size range
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d.  Tot. range
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1          34           4          30          24          28          26          16          20          12          14          16          36          34          38      23.714      10.314          44
     2          32          32          32          32          32          32          16          32          16          16          32          16          16          32      26.286       7.956          32
     3          20          36          22          16          16          16          22          38          38          14          14          14          16          42      23.143      10.487          46
     4           6           6          10          10          10           6           6           6           2           6           6           6          10           6       6.857       2.316          10
     5          13          16          18          13          14          16          16          23          16          20          40          16          16           6      17.357       7.551          40
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean      21.000      18.800      22.400      19.000      20.000      19.200      15.200      23.800      16.800      14.000      21.600      17.600      18.400      24.800      19.471       3.083      34.400
  s.d.      10.770      13.121       8.040       8.000       8.485       8.998       5.154      10.962      11.771       4.561      12.484       9.912       8.139      15.677       9.719       3.030      13.109
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
<a name="18_09_11at16_08_26_comp_sum_GW">&nbsp;</a>

----------------------
Garza-Williamson index
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Locus#     QBED1-1     QBED1-2     QBED2-1     QBED2-2      QLL1-1      QLL1-2      QLL3-1      QLL3-2      QRL1-1      QRL1-2      QRL2-1      QRL2-2      QTUR-1      QTUR-2        Mean        s.d.
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
     1     0.25714     0.60000     0.19355     0.28000     0.17241     0.18519     0.29412     0.19048     0.23077     0.26667     0.47059     0.24324     0.20000     0.20513     0.27066     0.12097
     2     0.15152     0.15152     0.12121     0.18182     0.15152     0.15152     0.23529     0.12121     0.17647     0.23529     0.15152     0.17647     0.17647     0.12121     0.16450     0.03639
     3     0.38095     0.27027     0.39130     0.47059     0.35294     0.47059     0.39130     0.30769     0.25641     0.46667     0.40000     0.40000     0.41176     0.23256     0.37165     0.07846
     4     0.57143     0.57143     0.45455     0.45455     0.45455     0.42857     0.57143     0.57143     0.66667     0.42857     0.57143     0.57143     0.45455     0.57143     0.52443     0.07525
     5     0.28571     0.35294     0.31579     0.35714     0.33333     0.35294     0.29412     0.20833     0.29412     0.33333     0.17073     0.29412     0.29412     0.42857     0.30824     0.06348
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Mean     0.32935     0.38923     0.29528     0.34882     0.29295     0.31776     0.35725     0.27983     0.32489     0.34611     0.35285     0.33705     0.30738     0.31178     0.32789     0.02973
  s.d.     0.14143     0.17297     0.12301     0.10835     0.11478     0.12813     0.11822     0.15752     0.17512     0.08954     0.16587     0.13808     0.11081     0.16433     0.13630     0.02723
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (18/09/11 at 16:27:27))
Total computing time for this run : 0h 19m 1s 704 ms
////////////////////////////////////////////////////////////////////


</pre>
</body></html>