
<html>
<!--Arlequin results file-->

<head>
<style>
BODY {background-color: white; font-size: 8pt; font-family: Courier New}
</style>
</head>

<body bgcolor=white>

<PRE>
<A NAME="15_09_11at19_19_24">&nbsp;</A>
////////////////////////////////////////////////////////////////////
RUN NUMBER 1 (15/09/11 at 19:19:24)
////////////////////////////////////////////////////////////////////

 #[WARNING # 1] : sum of genotype frequencies do not match specified sample size : adjusting sample size
<A NAME="15_09_11at19_19_24_run_information">&nbsp;</A>

Project information:
--------------------
	NbSamples      =  1
	DataType       =  MICROSAT
	GenotypicData  =  1
	GameticPhase   =  0
	RecessiveData  =  0

==============================
Settings used for Calculations
==============================

 General settings:
 -----------------
        Deletion Weight                = 1
        Transition Weight Weight       = 1
        Tranversion Weight Weight      = 1
        Epsilon Value                  = 1e-07
        Significant digits for output  = 5
        Use original haplotype definition
        Alllowed level of missing data = 0.3

 Active Tasks:
 -------------

    Standard indices:
    -----------------

    Molecular Diversity:
    --------------------
        Molecular Distance :No. of different alleles (FST)
        GammaA Value       = 0
        Theta estimators   : 

    Linkage disequilibrium test:
    ----------------------------
        No. of permutations          = 10000
        No. of Initial conditions    = 2
        Print Histogram and table : No

    Hardy-Weinberg equilibrium test:
    --------------------------------
        No. of steps in Markov chain      = 1000000
        No. of Dememorisation Steps       = 100000
        Required precision on Probability = 0
        Test association at the Locus level


    Correlation between distance matrices.
    --------------------------------------------------------

        No. of Permutations for significance = 1000
<A NAME="15_09_11at19_19_24_gen_struct">&nbsp;</A>
===============================================================================
== GENETIC STRUCTURE ANALYSIS
===============================================================================


Number of usable loci for distance computation : 9
Allowed level of missing data                  : 0.3

List of usable loci :
---------------------
   1     2     3     4     5     6     7     8     9  

List of loci with too much missing data :
-----------------------------------------

NONE

<A NAME="15_09_11at19_19_24_pairw_diff">&nbsp;</A>

======================================================================================
== Comparisons of pairs of population samples
======================================================================================

List of labels for population samples used below:
-------------------------------------------------

Label  	Population name
-----  	---------------
  1:	Everything


==============================================================================
== ANALYSES AT THE INTRA-POPULATION LEVEL
==============================================================================
<A NAME="15_09_11at19_19_24_samp0">&nbsp;</A>

===============================================================================
== Sample : 	Everything
===============================================================================


===============================
== Standard diversity indices : (Everything)
===============================

Reference: <i>Nei, M., 1987.</i>
Reference: <i>Garza, J.C., and Williamson, E.G. 2001.</i>
No. of gene copies       : 560
No. of loci              : 9
No. of usable loci       : 9 loci with less than 30.00 % missing data
No. of polymorphic loci  : 9


Results are only shown for polymorphic loci
---------------------------------------------------------------------
           Num.
           gene     Num.        Obs.      Exp.   Allelic        G-W
Locus#    copies   alleles      Het.       Het     range       stat.
---------------------------------------------------------------------
     1       560        20   0.31786   0.40447        44     0.44444
     2       560         6   0.52143   0.65234        32     0.18182
     3       560        24   0.66786   0.92591        48     0.48980
     4       550        21   0.62909   0.83779        46     0.44681
     5       560        24   0.66071   0.90660        54     0.43636
     6       550        13   0.22909   0.55406        58     0.22034
     7       556         5   0.50360   0.61788        10     0.45455
     8       554        11   0.53430   0.61735        40     0.26829
     9       550        16   0.46182   0.88191        34     0.45714
---------------------------------------------------------------------
  Mean   555.556    15.556   0.50286   0.71093    40.667     0.37773
  s.d.     4.667     7.265   0.15011   0.18344    13.466     0.11183
---------------------------------------------------------------------



================================
== Molecular diversity indices : (Everything)
================================

Reference: <i>Tajima, F., 1983.</i>
           <i>Tajima, F. 1993.</i>
           <i>Nei, M., 1987.</i>
           <i>Zouros, E., 1979.</i>
           <i>Ewens, W.J. 1972.</i>
Sample size                         : 560.00000
No. of haplotypes                   : 551

Allowed level of missing data       : 30.00000 %
Number of usable loci               : 9


Distance method                     : No. of different alleles
Average gene diversity over loci    :     0.699534 +/-     0.367608

(Standard deviations are for both the sampling and the stochastic processes)

====================================
== Pairwise linkage disequilibrium : (Everything)
====================================

Reference: <i>Slatkin, M. 1994a.</i>
           <i>Slatkin, M. and Excoffier, L. 1996.</i>
           <i>Lewontin, R. C., and K. Kojima 1960.</i>


Test of linkage disequilibrium for all pairs of loci:
-----------------------------------------------------


Permutation test using the EM algorithm
Number of permutations               : 10000
Number of initial conditions for EM  : 2

Pair(0, 1)
          LnLHood LD : -1107.86265          LnLHood LE : -1146.63150
          Exact P=  0.12861  +-   0.00330 (10100 permutations done)     Chi-square test value=77.53770 (P =  0.90403,  95 d.f.)
Pair(0, 2)
          LnLHood LD : -1869.47714          LnLHood LE : -1993.69948
          Exact P=  0.12535  +-   0.00333 (10100 permutations done)     Chi-square test value=248.44468 (P =  1.00000,  437 d.f.)
Pair(1, 2)
          LnLHood LD : -1963.29091          LnLHood LE : -2021.46160
          Exact P=  0.12733  +-   0.00357 (10100 permutations done)     Chi-square test value=116.34138 (P =  0.44748,  115 d.f.)
Pair(0, 3)
          LnLHood LD : -1552.41738          LnLHood LE : -1647.38269
          Exact P=  0.12505  +-   0.00311 (10100 permutations done)     Chi-square test value=189.93061 (P =  1.00000,  399 d.f.)
Pair(1, 3)
          LnLHood LD : -1623.19955          LnLHood LE : -1675.14481
          Exact P=  0.01802  +-   0.00112 (10100 permutations done)     Chi-square test value=103.89052 (P =  0.51227,  105 d.f.)
Pair(2, 3)
          LnLHood LD : -2352.32310          LnLHood LE : -2522.21279
          Exact P=  0.01406  +-   0.00129 (10100 permutations done)     Chi-square test value=339.77938 (P =  1.00000,  483 d.f.)
Pair(0, 4)
          LnLHood LD : -1789.55546          LnLHood LE : -1927.94453
          Exact P=  0.00020  +-   0.00014 (10100 permutations done)     Chi-square test value=276.77813 (P =  1.00000,  437 d.f.)
Pair(1, 4)
          LnLHood LD : -1897.67524          LnLHood LE : -1955.70665
          Exact P=  0.09257  +-   0.00308 (10100 permutations done)     Chi-square test value=116.06281 (P =  0.45470,  115 d.f.)
Pair(2, 4)
          LnLHood LD : -2568.08402          LnLHood LE : -2802.77463
          Exact P=  0.00218  +-   0.00052 (10100 permutations done)     Chi-square test value=469.38122 (P =  0.97035,  529 d.f.)
Pair(3, 4)
          LnLHood LD : -2289.85315          LnLHood LE : -2456.45784
          Exact P=  0.00970  +-   0.00097 (10100 permutations done)     Chi-square test value=333.20938 (P =  1.00000,  483 d.f.)
Pair(0, 5)
          LnLHood LD : -1114.11535          LnLHood LE : -1155.02187
          Exact P=  0.62703  +-   0.00470 (10100 permutations done)     Chi-square test value=81.81304 (P =  1.00000,  247 d.f.)
Pair(1, 5)
          LnLHood LD : -1160.14234          LnLHood LE : -1182.78399
          Exact P=  0.54386  +-   0.00478 (10100 permutations done)     Chi-square test value=45.28329 (P =  0.97018,  65 d.f.)
Pair(2, 5)
          LnLHood LD : -1939.88450          LnLHood LE : -2029.85197
          Exact P=  0.11238  +-   0.00319 (10100 permutations done)     Chi-square test value=179.93495 (P =  1.00000,  299 d.f.)
Pair(3, 5)
          LnLHood LD : -1601.79853          LnLHood LE : -1683.53518
          Exact P=  0.00238  +-   0.00051 (10100 permutations done)     Chi-square test value=163.47330 (P =  1.00000,  273 d.f.)
Pair(4, 5)
          LnLHood LD : -1863.17053          LnLHood LE : -1964.09702
          Exact P=  0.00149  +-   0.00041 (10100 permutations done)     Chi-square test value=201.85299 (P =  1.00000,  299 d.f.)
Pair(0, 6)
          LnLHood LD : -1101.19925          LnLHood LE : -1137.19770
          Exact P=  0.23475  +-   0.00452 (10100 permutations done)     Chi-square test value=71.99689 (P =  0.96222,  95 d.f.)
Pair(1, 6)
          LnLHood LD : -1155.77104          LnLHood LE : -1164.95982
          Exact P=  0.78099  +-   0.00423 (10100 permutations done)     Chi-square test value=18.37755 (P =  0.82595,  25 d.f.)
Pair(2, 6)
          LnLHood LD : -1953.35326          LnLHood LE : -2012.02780
          Exact P=  0.09653  +-   0.00265 (10100 permutations done)     Chi-square test value=117.34908 (P =  0.42162,  115 d.f.)
Pair(3, 6)
          LnLHood LD : -1620.74941          LnLHood LE : -1665.71101
          Exact P=  0.13614  +-   0.00334 (10100 permutations done)     Chi-square test value=89.92320 (P =  0.85273,  105 d.f.)
Pair(4, 6)
          LnLHood LD : -1894.30074          LnLHood LE : -1946.27285
          Exact P=  0.33139  +-   0.00476 (10100 permutations done)     Chi-square test value=103.94422 (P =  0.76097,  115 d.f.)
Pair(5, 6)
          LnLHood LD : -1148.28519          LnLHood LE : -1173.35019
          Exact P=  0.28406  +-   0.00432 (10100 permutations done)     Chi-square test value=50.13000 (P =  0.91305,  65 d.f.)
Pair(0, 7)
          LnLHood LD : -1152.94990          LnLHood LE : -1216.48676
          Exact P=  0.01297  +-   0.00111 (10100 permutations done)     Chi-square test value=127.07370 (P =  1.00000,  209 d.f.)
Pair(1, 7)
          LnLHood LD : -1226.90411          LnLHood LE : -1244.24887
          Exact P=  0.75069  +-   0.00415 (10100 permutations done)     Chi-square test value=34.68953 (P =  0.98533,  55 d.f.)
Pair(2, 7)
          LnLHood LD : -2005.68002          LnLHood LE : -2091.31686
          Exact P=  0.12703  +-   0.00301 (10100 permutations done)     Chi-square test value=171.27368 (P =  0.99998,  253 d.f.)
Pair(3, 7)
          LnLHood LD : -1678.19216          LnLHood LE : -1745.00007
          Exact P=  0.12624  +-   0.00336 (10100 permutations done)     Chi-square test value=133.61581 (P =  1.00000,  231 d.f.)
Pair(4, 7)
          LnLHood LD : -1955.30793          LnLHood LE : -2025.56191
          Exact P=  0.81000  +-   0.00381 (10100 permutations done)     Chi-square test value=140.50796 (P =  1.00000,  253 d.f.)
Pair(5, 7)
          LnLHood LD : -1214.08117          LnLHood LE : -1252.63924
          Exact P=  0.21386  +-   0.00415 (10100 permutations done)     Chi-square test value=77.11614 (P =  1.00000,  143 d.f.)
Pair(6, 7)
          LnLHood LD : -1213.68086          LnLHood LE : -1234.81507
          Exact P=  0.38871  +-   0.00464 (10100 permutations done)     Chi-square test value=42.26843 (P =  0.89576,  55 d.f.)
Pair(0, 8)
          LnLHood LD : -1711.48831          LnLHood LE : -1808.72456
          Exact P=  0.15495  +-   0.00356 (10100 permutations done)     Chi-square test value=194.47252 (P =  1.00000,  304 d.f.)
Pair(1, 8)
          LnLHood LD : -1785.34991          LnLHood LE : -1836.48668
          Exact P=  0.00743  +-   0.00092 (10100 permutations done)     Chi-square test value=102.27354 (P =  0.04733,  80 d.f.)
Pair(2, 8)
          LnLHood LD : -2478.30150          LnLHood LE : -2683.55467
          Exact P=  0.00000  +-   0.00000 (10100 permutations done)     Chi-square test value=410.50633 (P =  0.06244,  368 d.f.)
Pair(3, 8)
          LnLHood LD : -2196.64920          LnLHood LE : -2337.23788
          Exact P=  0.00069  +-   0.00025 (10100 permutations done)     Chi-square test value=281.17735 (P =  0.98666,  336 d.f.)
Pair(4, 8)
          LnLHood LD : -2427.75768          LnLHood LE : -2617.79972
          Exact P=  0.00000  +-   0.00000 (10100 permutations done)     Chi-square test value=380.08407 (P =  0.32091,  368 d.f.)
Pair(5, 8)
          LnLHood LD : -1749.32504          LnLHood LE : -1844.87705
          Exact P=  0.00000  +-   0.00000 (10100 permutations done)     Chi-square test value=191.10403 (P =  0.79362,  208 d.f.)
Pair(6, 8)
          LnLHood LD : -1776.88243          LnLHood LE : -1827.05288
          Exact P=  0.00782  +-   0.00088 (10100 permutations done)     Chi-square test value=100.34091 (P =  0.06169,  80 d.f.)
Pair(7, 8)
          LnLHood LD : -1822.18186          LnLHood LE : -1906.34194
          Exact P=  0.00000  +-   0.00000 (10100 permutations done)     Chi-square test value=168.32015 (P =  0.64794,  176 d.f.)


Histogram of the number of linked loci per locus

---------------------------------------------------------------------
Locus:     0    1    2    3    4    5    6    7    8
---------------------------------------------------------------------
           2    2    3    5    5    3    1    2    7
---------------------------------------------------------------------


Table of significant linkage disequilibrium (significance level=0.0500):

          ------------------------------------
Locus  # |  0|  1|  2|  3|  4|  5|  6|  7|  8|
          ------------------------------------
       0 |  *   -   -   -   +   -   -   +   - 
       1 |  -   *   -   +   -   -   -   -   + 
       2 |  -   -   *   +   +   -   -   -   + 
       3 |  -   +   +   *   +   +   -   -   + 
       4 |  +   -   +   +   *   +   -   -   + 
       5 |  -   -   -   +   +   *   -   -   + 
       6 |  -   -   -   -   -   -   *   -   + 
       7 |  +   -   -   -   -   -   -   *   + 
       8 |  -   +   +   +   +   +   +   +   * 

===============================
== Hardy-Weinberg equilibrium : (Everything)
===============================

Reference: <i>Guo, S. and Thompson, E. 1992.</i>
           <i>Levene H. (1949).</i>

Exact test using a Markov chain (for all Loci):
Forecasted chain length     :1000000
Dememorization steps        :100000

------------------------------------------------------------------------
Locus  #Genot       Obs.Het.     Exp.Het.   P-value     s.d.  Steps done
------------------------------------------------------------------------
    1     280       0.31786      0.40447    0.00000  0.00000     1001000
    2     280       0.52143      0.65234    0.00000  0.00000     1001000
    3     280       0.66786      0.92591    0.00000  0.00000     1001000
    4     275       0.62909      0.83779    0.00000  0.00000     1001000
    5     280       0.66071      0.90660    0.00000  0.00000     1001000
    6     275       0.22909      0.55406    0.00000  0.00000     1001000
    7     278       0.50360      0.61788    0.00000  0.00000     1001000
    8     277       0.53430      0.61735    0.00114  0.00003     1001000
    9     275       0.46182      0.88191    0.00000  0.00000     1001000
------------------------------------------------------------------------




================================================================================
== Summary of computations done within populations
================================================================================

<A NAME="15_09_11at19_19_24_comp_sum_Basic">&nbsp;</A>

----------------
Basic properties
--------------------------------------------------------
          Statistics  Everything        Mean        s.d.
--------------------------------------------------------
  No. of gene copies         560     560.000       0.000
         No. of loci           9       9.000       0.000
  No. of usable loci           9       9.000       0.000
  No. of polym. loci           9       9.000       0.000
--------------------------------------------------------
<A NAME="15_09_11at19_19_24_comp_sum_het">&nbsp;</A>

-----------------------
Expected heterozygosity
------------------------------------------------------
Locus#  Everything        Mean        s.d.   Tot. Het.
------------------------------------------------------
     1     0.40447     0.40447     0.00000     0.40447
     2     0.65234     0.65234     0.00000     0.65234
     3     0.92591     0.92591     0.00000     0.92591
     4     0.83779     0.83779     0.00000     0.83779
     5     0.90660     0.90660     0.00000     0.90660
     6     0.55406     0.55406     0.00000     0.55406
     7     0.61788     0.61788     0.00000     0.61788
     8     0.61735     0.61735     0.00000     0.61735
     9     0.88191     0.88191     0.00000     0.88191
------------------------------------------------------
  Mean     0.71093     0.71093     0.00000     0.71093
  s.d.     0.18344     0.18344     0.00000     0.17295
------------------------------------------------------
<A NAME="15_09_11at19_19_24_comp_sum_thetaH">&nbsp;</A>

------------------------------------------
Theta(H) under the stepwise mutation model
------------------------------------------
Locus#  Everything        Mean        s.d.
------------------------------------------
     1     1.57577     1.57577     0.00000
     2     1.70467     1.70467     0.00000
     3     6.78889     6.78889     0.00000
     4     3.17931     3.17931     0.00000
     5     5.40482     5.40482     0.00000
     6     1.52366     1.52366     0.00000
     7     1.61771     1.61771     0.00000
     8     1.61660     1.61660     0.00000
     9     4.30093     4.30093     0.00000
------------------------------------------
  Mean     1.93297     1.93297     0.00000
------------------------------------------
<A NAME="15_09_11at19_19_24_comp_sum_numAll">&nbsp;</A>

-----------------
Number of alleles
------------------------------------------------------
Locus#  Everything        Mean        s.d. Tot. number
------------------------------------------------------
     1          20      20.000       0.000          20
     2           6       6.000       0.000           6
     3          24      24.000       0.000          24
     4          21      21.000       0.000          21
     5          24      24.000       0.000          24
     6          13      13.000       0.000          13
     7           5       5.000       0.000           5
     8          11      11.000       0.000          11
     9          16      16.000       0.000          16
------------------------------------------------------
  Mean      15.556      15.556       0.000      15.556
  s.d.       7.265       7.265       0.000       6.849
------------------------------------------------------
<A NAME="15_09_11at19_19_24_comp_sum_allRange">&nbsp;</A>

------------------
Allelic size range
------------------------------------------------------
Locus#  Everything        Mean        s.d.  Tot. range
------------------------------------------------------
     1          44      44.000       0.000          44
     2          32      32.000       0.000          32
     3          48      48.000       0.000          48
     4          46      46.000       0.000          46
     5          54      54.000       0.000          54
     6          58      58.000       0.000          58
     7          10      10.000       0.000          10
     8          40      40.000       0.000          40
     9          34      34.000       0.000          34
------------------------------------------------------
  Mean      40.667      40.667       0.000      40.667
  s.d.      13.466      13.466       0.000      13.466
------------------------------------------------------
<A NAME="15_09_11at19_19_24_comp_sum_GW">&nbsp;</A>

----------------------
Garza-Williamson index
------------------------------------------
Locus#  Everything        Mean        s.d.
------------------------------------------
     1     0.44444     0.44444     0.00000
     2     0.18182     0.18182     0.00000
     3     0.48980     0.48980     0.00000
     4     0.44681     0.44681     0.00000
     5     0.43636     0.43636     0.00000
     6     0.22034     0.22034     0.00000
     7     0.45455     0.45455     0.00000
     8     0.26829     0.26829     0.00000
     9     0.45714     0.45714     0.00000
------------------------------------------
  Mean     0.37773     0.37773     0.00000
  s.d.     0.11183     0.11183     0.11183
------------------------------------------

////////////////////////////////////////////////////////////////////
END OF RUN NUMBER 1 (15/09/11 at 21:37:10))
Total computing time for this run : 2h 17m 45s 953 ms
////////////////////////////////////////////////////////////////////


</body>
</PRE>
</html>

